<html>
<head>
  <title>I-TASSER results</title>
</head>

<script type="text/javascript" src="../../bin/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="../../bin/jmol/Jmol2.js"></script>
<body bgcolor="#FFFFFF">
<font face="Times New Roman">
[<a href="http://zhanglab.ccmb.med.umich.edu">Home</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]
<br/>
<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">
<tr><td>
  <h1 align="center">I-TASSER results for job id S552663</h1>
<center> (Click on <a href="S552663_results.tar.bz2" download>S552663_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page. Click on <a href=http://zhanglab.ccmb.med.umich.edu/I-TASSER/annotation target=_blank>Annotation of I-TASSER Output</a> to read the instructions for how to interpret the results on this page. Model results are kept on the server for 60 days, there is no way to retrieve the modeling data older than 2 months)
</center><br>
</td></tr>
<tr><td>
   <div style="background:#6599FF;width:350px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence in <a href=http://zhanglab.ccmb.med.umich.edu/FASTA/ target="_blank">FASTA format</a></font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
   <tr><td colspan="2">>protein<br>
MGVDFLTKTVTISDKTSKLQLWDTGGQEFVKALLPFYFTGAAGGALVFDLTNRNSFNSLG<br/>
YWLKQIRKNSGDIPVVLAGNKVDLATQRKVTTEEGQLFASKNNLLYLETSAKTGVSVPDL<br/>
FKSLVQIIVEEQEKKEEEESKEKSQESEATDPEMEEQTFFKRI<br/>
</td></tr>
   </table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;<br/>MGVDFLTKTVTISDKTSKLQLWDTGGQEFVKALLPFYFTGAAGGALVFDLTNRNSFNSLGYWLKQIRKNSGDIPVVLAGNKVDLATQRKVTTEEGQLFASKNNLLYLETSAKTGVSVPDLFKSLVQIIVEEQEKKEEEESKEKSQESEATDPEMEEQTFFKRI</td></tr>
    <tr><td><font="Arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">CCC<font color="blue">SSSSSSSSS</font>CC<font color="blue">SSSSSSSSS</font>CCC<font color="red">HHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSS</font>CCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSS</font>CCCC<font color="red">HHH</font>CCCC<font color="red">HHHHHHHHHHH</font>CC<font color="blue">SSSS</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCCCCCCCCCCCCCCCCCCCCCCCCCCCCC</td>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font></td><td BGCOLOR="#ECECEC">9457678899989999999886088567766657877667879999997989899888999999999848997499999883027548868999999999809989987589994999999999999998664266677766767666779873778688859</td></tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC"><font color=red>H</font>:Helix; <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;<br/>MGVDFLTKTVTISDKTSKLQLWDTGGQEFVKALLPFYFTGAAGGALVFDLTNRNSFNSLGYWLKQIRKNSGDIPVVLAGNKVDLATQRKVTTEEGQLFASKNNLLYLETSAKTGVSVPDLFKSLVQIIVEEQEKKEEEESKEKSQESEATDPEMEEQTFFKRI</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">7232232220407644010001120321322211322233020000000013461163044005203721430000000012216663414374035107736030000003464203400340044027316754454565555545446364767444542</td><tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>
   </td></tr></table>
   </div>
   <br/>

<font size=2>
(B-factor is a value to indicate the extent of the inherent thermal 

mobility of residues/atoms in proteins. In I-TASSER, this value is 

deduced from threading template proteins from the PDB in combination with the 

sequence profiles derived from sequence databases. The reported B-factor 

profile in the figure below corresponds to the normalized B-factor of the 

target protein, defined by B=(B'-u)/s, where B' is the raw B-factor 

value, u and s are respectively the mean and standard deviation of the raw 

B-factors along the sequence. 

<a href="lscore.txt" target="_blank"  >Click here to read more about predicted 

normalized B-factor</a>) 

</font> 

<br/> 

   <br/> 

   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr> <td align =center > <img src=BFP.png>  </td>  </tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 threading templates used by I-TASSER</font>
   </td></tr></table>
   </div>
   <br/>
   
<font size=2> 

(I-TASSER modeling starts from the structure templates identified by LOMETS from the PDB library. 

LOMETS is a meta-server threading approach containing multiple threading programs, 

where each threading program can generate tens of thousands of 

template alignments. I-TASSER only uses the templates of the highest significance in the threading alignments, 

the significance of which are measured by the Z-score, i.e. the difference between the raw and average scores 

in the unit of standard deviation. The templates in this section are the 10 best templates selected from 

the LOMETS threading programs. Usually, one template of the highest Z-score is selected from each threading program, where the threading programs are sorted by the average performance in the large-scale benchmark test experiments.) 

</font> 

   <br/> 

   <br/> 

   <div style="position:relative;left:30px;">
   <table style="font-size:13px;background:#F2F2F2;">
      <tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Z-score</font></th><th valign="top"><font face="Arial">Download<br/>Align.</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;</td></tr>
      <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">Sec.Str<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCC<font color="blue">SSSSSSSSS</font>CC<font color="blue">SSSSSSSSS</font>CCC<font color="red">HHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSS</font>CCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCCC<font color="blue">SSSSSSS</font>CCCC<font color="red">HHH</font>CCCC<font color="red">HHHHHHHHHHH</font>CC<font color="blue">SSSS</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCCCCCCCCCCCCCCCCCCCCCCCCCCCCC<br/>MGVDFLTKTVTISDKTSKLQLWDTGGQEFVKALLPFYFTGAAGGALVFDLTNRNSFNSLGYWLKQIRKNSGDIPVVLAGNKVDLATQRKVTTEEGQLFASKNNLLYLETSAKTGVSVPDLFKSLVQIIVEEQEKKEEEESKEKSQESEATDPEMEEQTFFKRI</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></td><td align ="center"><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 3.51</font></td><td align="center"><a href="threading1.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font><font color="#0F820F">V</font><font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELDG<font color="#145AFF">K</font><font color="#FA9600">T</font>V<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFRTITSS<font color="#3232AA">Y</font>YR<font color="#666666">G</font>SH<font color="#666666">G</font>III<font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>V<font color="#FA9600">T</font>DQE<font color="#FA9600">S</font><font color="#3232AA">F</font><font color="#00DCDC">N</font>GVKM<font color="#B45AB4">W</font><font color="#0F820F">L</font>QE<font color="#0F820F">I</font>DRYATTVLKL<font color="#0F820F">L</font>V<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font><font color="#0F820F">L</font>KDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>EYDVAKE<font color="#3232AA">F</font><font color="#333333">A</font>DA<font color="#00DCDC">N</font>KMPF<font color="#0F820F">L</font><font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font>LDSTN<font color="#0F820F">V</font>E<font color="#E60A0A">D</font>A<font color="#3232AA">F</font>LTMARQ<font color="#0F820F">I</font>K<font color="#E60A0A">E</font>SMSQQNLN<font color="#E60A0A">E</font>TTQ<font color="#145AFF">K</font>KEDKGNVNLKGQ-----SLC</td></tr>
       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2a5j" target="_blank">2a5jA</a></td><td align ="center"><font face="Arial"> 0.40</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.84</font></td><td align="center"><font face="Arial"> 3.24</font></td><td align="center"><a href="threading2.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font><font color="#0F820F">V</font>E<font color="#3232AA">F</font>GARM<font color="#0F820F">V</font>N<font color="#0F820F">I</font>DG<font color="#145AFF">K</font>QI<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>SFRSITRS<font color="#3232AA">Y</font>YR<font color="#666666">G</font><font color="#333333">A</font><font color="#333333">A</font><font color="#666666">G</font>AL<font color="#0F820F">L</font><font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>I<font color="#FA9600">T</font>R<font color="#145AFF">R</font>ET<font color="#3232AA">F</font><font color="#00DCDC">N</font>H<font color="#0F820F">L</font>TS<font color="#B45AB4">W</font><font color="#0F820F">L</font>EDA<font color="#145AFF">R</font>QH<font color="#FA9600">S</font>SNMVIM<font color="#0F820F">L</font>I<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>S<font color="#E60A0A">D</font><font color="#0F820F">L</font>ESR<font color="#145AFF">R</font>D<font color="#0F820F">V</font>KR<font color="#E60A0A">E</font><font color="#E60A0A">E</font><font color="#666666">G</font>EA<font color="#3232AA">F</font><font color="#333333">A</font>REHG<font color="#0F820F">L</font>IFM<font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font><font color="#145AFF">K</font><font color="#FA9600">T</font>ACN<font color="#0F820F">V</font>EEA<font color="#3232AA">F</font>INTAKE<font color="#0F820F">I</font>YRKIQQGLF--------------------------</td></tr>
       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></td><td align ="center"><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 3.07</font></td><td align="center"><a href="threading3.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font><font color="#0F820F">V</font><font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELDG<font color="#145AFF">K</font><font color="#FA9600">T</font>V<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFRTITSS<font color="#3232AA">Y</font>YR<font color="#666666">G</font>SH<font color="#666666">G</font>III<font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>V<font color="#FA9600">T</font>DQE<font color="#FA9600">S</font><font color="#3232AA">F</font><font color="#00DCDC">N</font>GVKM<font color="#B45AB4">W</font><font color="#0F820F">L</font>QE<font color="#0F820F">I</font>DRYATTVLKL<font color="#0F820F">L</font>V<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font><font color="#0F820F">L</font>KDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>EYDVAKE<font color="#3232AA">F</font><font color="#333333">A</font>DA<font color="#00DCDC">N</font>KMPF<font color="#0F820F">L</font><font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font>LDSTN<font color="#0F820F">V</font>E<font color="#E60A0A">D</font>A<font color="#3232AA">F</font>LTMARQ<font color="#0F820F">I</font>K<font color="#E60A0A">E</font>SMSQQNLN<font color="#E60A0A">E</font>TTQ<font color="#145AFF">K</font>KEDKGNVNLKGQSLC-----</td></tr>
       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5lpn" target="_blank">5lpn</a></td><td align ="center"><font face="Arial"> 0.36</font></td><td><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.82</font></td><td align="center"><font face="Arial"> 0.94</font></td><td align="center"><a href="threading4.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font>I<font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELQG<font color="#145AFF">K</font>KI<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFHTITTS<font color="#3232AA">Y</font>YR<font color="#666666">G</font><font color="#333333">A</font>M<font color="#666666">G</font>IM<font color="#0F820F">L</font><font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>I<font color="#FA9600">T</font><font color="#00DCDC">N</font>GK<font color="#FA9600">S</font><font color="#3232AA">F</font>ENISK<font color="#B45AB4">W</font><font color="#0F820F">L</font>RN<font color="#0F820F">I</font>DEHAE<font color="#E60A0A">D</font>VERM<font color="#0F820F">L</font>L<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font>MDDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>PKGK<font color="#666666">G</font>EQI<font color="#333333">A</font>REHGIRFF<font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font><font color="#145AFF">K</font>ANINIEKA<font color="#3232AA">F</font>LT<font color="#0F820F">L</font>AED<font color="#0F820F">I</font>LRKTP------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5lpn" target="_blank">5lpn</a></td><td align ="center"><font face="Arial"> 0.36</font></td><td><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.82</font></td><td align="center"><font face="Arial"> 0.75</font></td><td align="center"><a href="threading5.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font>I<font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELQG<font color="#145AFF">K</font>KI<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFHTITTS<font color="#3232AA">Y</font>YR<font color="#666666">G</font><font color="#333333">A</font>M<font color="#666666">G</font>IM<font color="#0F820F">L</font><font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>I<font color="#FA9600">T</font><font color="#00DCDC">N</font>GK<font color="#FA9600">S</font><font color="#3232AA">F</font>ENISK<font color="#B45AB4">W</font><font color="#0F820F">L</font>RN<font color="#0F820F">I</font>DEHAE<font color="#E60A0A">D</font>VERM<font color="#0F820F">L</font>L<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font>MDDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>PKGK<font color="#666666">G</font>EQI<font color="#333333">A</font>REHGIRFF<font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font><font color="#145AFF">K</font>ANINIEKA<font color="#3232AA">F</font>LT<font color="#0F820F">L</font>AED<font color="#0F820F">I</font>LRKTP------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></td><td align ="center"><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 3.14</font></td><td align="center"><a href="threading6.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font><font color="#0F820F">V</font><font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELDG<font color="#145AFF">K</font><font color="#FA9600">T</font>V<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFRTITSS<font color="#3232AA">Y</font>YR<font color="#666666">G</font>SH<font color="#666666">G</font>III<font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>V<font color="#FA9600">T</font>DQE<font color="#FA9600">S</font><font color="#3232AA">F</font><font color="#00DCDC">N</font>GVKM<font color="#B45AB4">W</font><font color="#0F820F">L</font>QE<font color="#0F820F">I</font>DRYATTVLKL<font color="#0F820F">L</font>V<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font><font color="#0F820F">L</font>KDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>EYDVAKE<font color="#3232AA">F</font><font color="#333333">A</font>DA<font color="#00DCDC">N</font>KMPF<font color="#0F820F">L</font><font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font>LDSTN<font color="#0F820F">V</font>E<font color="#E60A0A">D</font>A<font color="#3232AA">F</font>LTMARQ<font color="#0F820F">I</font>K<font color="#E60A0A">E</font>SMSQQNLN<font color="#E60A0A">E</font>TTQ<font color="#145AFF">K</font>KEDKGNVNLKGQ-----SLC</td></tr>
       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5lpn" target="_blank">5lpn</a></td><td align ="center"><font face="Arial"> 0.37</font></td><td><font face="Arial"> 0.30</font></td><td><font face="Arial"> 0.80</font></td><td align="center"><font face="Arial"> 1.10</font></td><td align="center"><a href="threading7.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font>I<font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELQG<font color="#145AFF">K</font>KI<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFHTITTS<font color="#3232AA">Y</font>YR<font color="#666666">G</font><font color="#333333">A</font>M<font color="#666666">G</font>IM<font color="#0F820F">L</font><font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>I<font color="#FA9600">T</font><font color="#00DCDC">N</font>GK<font color="#FA9600">S</font><font color="#3232AA">F</font>ENISK<font color="#B45AB4">W</font><font color="#0F820F">L</font>RN<font color="#0F820F">I</font>DEHAE<font color="#E60A0A">D</font>VERM<font color="#0F820F">L</font>L<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font>MDDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>PKGK<font color="#666666">G</font>EQI<font color="#333333">A</font>REHGIRFF<font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font><font color="#145AFF">K</font>ANINIEKA<font color="#3232AA">F</font>LT<font color="#0F820F">L</font>AED<font color="#0F820F">I</font>LRK--------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5ub8" target="_blank">5ub8A</a></td><td align ="center"><font face="Arial"> 0.40</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.85</font></td><td align="center"><font face="Arial"> 3.91</font></td><td align="center"><a href="threading8.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">E<font color="#666666">G</font><font color="#0F820F">V</font>E<font color="#3232AA">F</font>A<font color="#FA9600">T</font>R<font color="#FA9600">T</font>LE<font color="#0F820F">I</font>DG<font color="#145AFF">K</font>RV<font color="#145AFF">K</font>A<font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RYR<font color="#333333">A</font>ITSA<font color="#3232AA">Y</font>YR<font color="#666666">G</font><font color="#333333">A</font>V<font color="#666666">G</font>ALI<font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>IAKTE<font color="#FA9600">S</font>YE<font color="#FA9600">S</font>VSR<font color="#B45AB4">W</font><font color="#0F820F">L</font><font color="#145AFF">K</font>ELKEHADN<font color="#0F820F">I</font>IIE<font color="#0F820F">L</font>V<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>S<font color="#E60A0A">D</font><font color="#0F820F">L</font>DHL<font color="#145AFF">R</font>A<font color="#0F820F">V</font>P<font color="#FA9600">T</font><font color="#E60A0A">E</font><font color="#E60A0A">E</font>AKN<font color="#3232AA">F</font><font color="#333333">A</font>ME<font color="#00DCDC">N</font><font color="#00DCDC">N</font><font color="#0F820F">L</font><font color="#0F820F">L</font>FT<font color="#E60A0A">E</font>A<font color="#FA9600">S</font><font color="#333333">A</font>LSSDN<font color="#0F820F">V</font>DLS<font color="#3232AA">F</font>HQ<font color="#0F820F">L</font>LKN<font color="#0F820F">I</font>Y<font color="#E60A0A">E</font>MIS<font color="#145AFF">K</font>HQL<font color="#E60A0A">E</font>-------------------------</td></tr>
       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></td><td align ="center"><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.34</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 2.36</font></td><td align="center"><a href="threading9.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font><font color="#0F820F">V</font><font color="#E60A0A">D</font><font color="#3232AA">F</font>KI<font color="#145AFF">K</font><font color="#FA9600">T</font><font color="#0F820F">V</font>ELDG<font color="#145AFF">K</font><font color="#FA9600">T</font>V<font color="#145AFF">K</font><font color="#0F820F">L</font><font color="#00DCDC">Q</font>I<font color="#B45AB4">W</font><font color="#E60A0A">D</font><font color="#FA9600">T</font>A<font color="#666666">G</font><font color="#00DCDC">Q</font><font color="#E60A0A">E</font>RFRTITSS<font color="#3232AA">Y</font>YR<font color="#666666">G</font>SH<font color="#666666">G</font>III<font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>V<font color="#FA9600">T</font>DQE<font color="#FA9600">S</font><font color="#3232AA">F</font><font color="#00DCDC">N</font>GVKM<font color="#B45AB4">W</font><font color="#0F820F">L</font>QE<font color="#0F820F">I</font>DRATSTVLKL<font color="#0F820F">L</font>V<font color="#666666">G</font><font color="#00DCDC">N</font><font color="#145AFF">K</font>C<font color="#E60A0A">D</font><font color="#0F820F">L</font>KDK<font color="#145AFF">R</font>V<font color="#0F820F">V</font>EYDVAKE<font color="#3232AA">F</font><font color="#333333">A</font>DA<font color="#00DCDC">N</font>KMPF<font color="#0F820F">L</font><font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font>LDSTN<font color="#0F820F">V</font>E<font color="#E60A0A">D</font>A<font color="#3232AA">F</font>LTMARQ<font color="#0F820F">I</font>K<font color="#E60A0A">E</font>SMSQQNLN<font color="#E60A0A">E</font>TTQ<font color="#145AFF">K</font>KEDKGNVNLKGQSLC-----</td></tr>
       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=6jmg" target="_blank">6jmgA</a></td><td align ="center"><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.89</font></td><td align="center"><font face="Arial"> 5.37</font></td><td align="center"><a href="threading10.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">I<font color="#666666">G</font>I<font color="#E60A0A">D</font>YGVTK<font color="#0F820F">V</font>H<font color="#0F820F">I</font>K<font color="#E60A0A">D</font>REI<font color="#145AFF">K</font>VNIF<font color="#E60A0A">D</font>MA<font color="#666666">G</font>HP<font color="#3232AA">F</font>FYEVRNEFYKDTQ<font color="#666666">G</font>VI<font color="#0F820F">L</font><font color="#0F820F">V</font>Y<font color="#E60A0A">D</font>VGHKET<font color="#3232AA">F</font>E<font color="#FA9600">S</font><font color="#0F820F">L</font>DG<font color="#B45AB4">W</font><font color="#0F820F">L</font>AEMKQEGDN<font color="#0F820F">I</font>VFAVCA<font color="#00DCDC">N</font><font color="#145AFF">K</font>I<font color="#E60A0A">D</font>STKH<font color="#145AFF">R</font>S<font color="#0F820F">V</font>DES<font color="#E60A0A">E</font><font color="#666666">G</font>R<font color="#0F820F">L</font>WSESKGF<font color="#0F820F">L</font><font color="#3232AA">Y</font>F<font color="#E60A0A">E</font><font color="#FA9600">T</font><font color="#FA9600">S</font><font color="#333333">A</font>QS<font color="#666666">G</font>EGINEM<font color="#3232AA">F</font>QAFYSA<font color="#0F820F">I</font><font color="#0F820F">V</font>DLCDNGGKRPVSAINI------------------</td></tr>
      <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score >1 mean a good alignment and vice versa.</td></tr>
        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>
        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>
        <tr><td>&nbsp;</td><td align ="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
1: MUSTER&nbsp;&nbsp;&nbsp;2: FFAS-3D&nbsp;&nbsp;&nbsp;3: SPARKS-X&nbsp;&nbsp;&nbsp;4: HHSEARCH2&nbsp;&nbsp;&nbsp;5: HHSEARCH I&nbsp;&nbsp;&nbsp;6: Neff-PPAS&nbsp;&nbsp;&nbsp;7: HHSEARCH&nbsp;&nbsp;&nbsp;8: pGenTHREADER&nbsp;&nbsp;&nbsp;9: PROSPECT2&nbsp;&nbsp;&nbsp;10: PRC&nbsp;&nbsp;&nbsp;        </td></tr>
      </table>
      </td></tr></table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:380px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Top 5 final models predicted by I-TASSER </font>
   </td></tr></table>
   </div>
   <br/>
  
<font size=2>

(For each target, I-TASSER simulations generate a large ensemble of
structural conformations, called decoys. To select the final models, I-TASSER uses the SPICKER program to cluster
all the decoys based on the pair-wise structure similarity, and reports up to five models which
corresponds to the five largest structure clusters. The confidence of each model is quantitatively
measured by C-score that is calculated based on the significance of threading template alignments
and the convergence parameters of the structure assembly simulations.
C-score is typically in the range of [-5, 2], where a C-score of a higher value signifies a model with a higher 
confidence and vice-versa. TM-score and RMSD are estimated based on C-score and protein length following 
the correlation observed between these qualities. Since the top 5 models are ranked by the cluster size,
it is possible that the lower-rank models have a higher C-score in 
rare cases. Although the first model has a better
quality in most cases, it is also possible that the lower-rank models have a better quality than the
higher-rank models as seen in our benchmark tests. If the I-TASSER simulations converge, it
is possible to have less than 5 clusters generated; this is usually an indication that the
models have a good quality because of the converged simulations.)
<ul>
<li><a href=cscore.txt target="_blank">More about C-score</a>
<li><a href=lscore.html>Local structure accuracy profile of the top five models</a>
</ul>
</font>
<ul>   <div style="position:relative;left:1px;">
       <table  style="table-layout:fixed;width:1780px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
  <tr> <td colspan=5> <font size=2>(By right-click on the images, you can export image file or change the configurations, e.g. modifying the background color or stopping the spin of
your models)</font></td></tr>
       <tr>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model2.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model3.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model4.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model5.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       </tr> <tr> <td  align =left>
<li><a href="model1.pdb" download>Download Model 1</a>

<li>C-score=0.34 (<a href="cscore.txt" target="_blank" >Read more about C-score</a>) 

<li>Estimated TM-score = 0.76&plusmn;0.10

<li>Estimated RMSD = 4.2&plusmn;2.8&Aring;

<br><br>

</td>
 <td  align =left>
<li><a href="model2.pdb" download>Download Model 2</a>

<li>C-score = -3.51 <br><br>

</td>
 <td  align =left>
<li><a href="model3.pdb" download>Download Model 3</a>

<li>C-score = -4.16 <br><br>

</td>
 <td  align =left>
<li><a href="model4.pdb" download>Download Model 4</a>

<li>C-score = -4.74 <br><br>

</td>
 <td  align =left>
<li><a href="model5.pdb" download>Download Model 5</a>

<li>C-score = -4.75 <br><br>

</td>
       </tr>
       </table>
    </div>
    </div>
  </td></tr>
  </ul>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:670px;">
      <table>
         <tr><td style= "valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins structurally close to the target in the PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>
      </table>
   </div>
   <br/>
   
<font size=2>

(After the structure assembly simulation, I-TASSER uses the TM-align structural alignment program to match the first I-TASSER model to all structures in the PDB library. This section reports the top 10 proteins from the PDB that have the closest structural similarity, 

i.e. the highest <a href=http://zhanglab.ccmb.med.umich.edu/TM-score>TM-score</a>, to the predicted I-TASSER model. Due to the structural similarity, these proteins often have similar function to the target. However, users are encouraged to use the data in the next section 'Predicted function using COACH' to infer the function of the target protein, since COACH has been extensively trained to derive biological functions from multi-source of sequence and structure features which has on average a higher accuracy than the function annotations derived only from the global structure comparison.)</font> 

   <br/>

   <br/>

   <div>
   <div style="position:relative;left:30px;background:#F2F2F2;">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
           <tr>
           <td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1_1ukvY.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br/><br/>
                 <table border="0" style="font-family:Arial;font-size:13px;">
                    <tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th >Rank</th><th >PDB Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th>Cov</th><th>Alignment</th></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1ukvY.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script></td><font face="Monospace">
                       <td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></font></td><td align="center">0.913</td><td valign="middle" align="center"> 1.66</td><td valign="middle" align="center">0.335</td><td valign="middle">0.969</td><td align="center"><a href="model1_1ukvY.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2bmeA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2bme" target="_blank">2bmeA</a></font></td><td align="center">0.791</td><td valign="middle" align="center"> 1.64</td><td valign="middle" align="center">0.412</td><td valign="middle">0.834</td><td align="center"><a href="model1_2bmeA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2f9mA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2f9m" target="_blank">2f9mA</a></font></td><td align="center">0.788</td><td valign="middle" align="center"> 1.98</td><td valign="middle" align="center">0.357</td><td valign="middle">0.853</td><td align="center"><a href="model1_2f9mA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2o52A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2o52" target="_blank">2o52A</a></font></td><td align="center">0.787</td><td valign="middle" align="center"> 1.79</td><td valign="middle" align="center">0.417</td><td valign="middle">0.847</td><td align="center"><a href="model1_2o52A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2gf9A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2gf9" target="_blank">2gf9A</a></font></td><td align="center">0.787</td><td valign="middle" align="center"> 1.22</td><td valign="middle" align="center">0.388</td><td valign="middle">0.822</td><td align="center"><a href="model1_2gf9A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5lpnA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5lpn" target="_blank">5lpnA</a></font></td><td align="center">0.785</td><td valign="middle" align="center"> 1.03</td><td valign="middle" align="center">0.361</td><td valign="middle">0.816</td><td align="center"><a href="model1_5lpnA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2a5jA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2a5j" target="_blank">2a5jA</a></font></td><td align="center">0.782</td><td valign="middle" align="center"> 1.74</td><td valign="middle" align="center">0.412</td><td valign="middle">0.834</td><td align="center"><a href="model1_2a5jA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2oilA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2oil" target="_blank">2oilA</a></font></td><td align="center">0.776</td><td valign="middle" align="center"> 1.40</td><td valign="middle" align="center">0.381</td><td valign="middle">0.822</td><td align="center"><a href="model1_2oilA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3bc1A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bc1" target="_blank">3bc1A</a></font></td><td align="center">0.776</td><td valign="middle" align="center"> 1.36</td><td valign="middle" align="center">0.370</td><td valign="middle">0.828</td><td align="center"><a href="model1_3bc1A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3bfkB.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bfk" target="_blank">3bfkB</a></font></td><td align="center">0.774</td><td valign="middle" align="center"> 1.99</td><td valign="middle" align="center">0.345</td><td valign="middle">0.847</td><td align="center"><a href="model1_3bfkB.pdb" download><font face="Arial">Download</font></a></td></tr>
               </table><br/>
               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
                <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
                <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
               </table>
          </td></tr>
       </table>
    </div>
    </div>
  </td></tr>
  <tr><td>
  <br/><br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted function using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a> and <a href="http://zhanglab.ccmb.med.umich.edu/COACH" target="_blank">COACH</a></font>
   </td></tr></table>
   </div>
  <br/>
  
<font size=2>

(This section reports biological annotations of the target 
protein by COFACTOR and COACH based on the I-TASSER structure prediction. 
While COFACTOR deduces protein functions (ligand-binding sites, EC and GO)
using structure comparison and protein-protein networks,
COACH is a meta-server approach that combines multiple function
annotation results (on ligand-binding sites) from the COFACTOR, 
TM-SITE and S-SITE programs.)

</font>

<br/>

<br/>

  <tr><td>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Ligand binding sites</font>
      </td></tr></table>
   </div>
  <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td valign="top" align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load CH_TM_1z0aA_BS02_GDP.pdb; calculate structure rama; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 80,81,83,84,110,111,112; wireframe 75; spacefill 100; color blue;select 80 and *.ca; label %m%R; color label magenta;select 81 and *.ca; label %m%R; color label magenta;select 83 and *.ca; label %m%R; color label magenta;select 84 and *.ca; label %m%R; color label magenta;select 110 and *.ca; label %m%R; color label magenta;select 111 and *.ca; label %m%R; color label magenta;select 112 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;     set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT> jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT>
</td>
       <td valign="top">       <table border="0"  style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>C-score</th><th>Cluster<br>size</th> <th>PDB<br>Hit</th> <th>Lig<br>Name</th> <th>Download<br>Complex</th> <th align=left>Ligand Binding Site Residues</th> </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_1z0aA_BS02_GDP.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 80,81,83,84,110,111,112; color blue; wireframe 75; spacefill 100;select 80 and *.ca; label %m%R; color label magenta;select 81 and *.ca; label %m%R; color label magenta;select 83 and *.ca; label %m%R; color label magenta;select 84 and *.ca; label %m%R; color label magenta;select 110 and *.ca; label %m%R; color label magenta;select 111 and *.ca; label %m%R; color label magenta;select 112 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", true);</script></td>
    <font face="Monospace"><td valign="middle" align="center">1</td><td valign="middle" align="center">0.61</td> <td valign="middle" align="center">293</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=1z0a target="_blank">1z0aA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=GDP target="_blank">GDP</a> </td>  <td valign="middle" align=center><a href="CH_TM_1z0aA_BS02_GDP.pdb" download>Rep</a>, <a href="CH_complex1.pdb" download>Mult</a></td>  <td valign="middle" align="left">80,81,83,84,110,111,112</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_1nvwQ_BS02_GNP.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 25,26,27,80,81,83,84,110,111,112; color blue; wireframe 75; spacefill 100;select 25 and *.ca; label %m%R; color label magenta;select 26 and *.ca; label %m%R; color label magenta;select 27 and *.ca; label %m%R; color label magenta;select 80 and *.ca; label %m%R; color label magenta;select 81 and *.ca; label %m%R; color label magenta;select 83 and *.ca; label %m%R; color label magenta;select 84 and *.ca; label %m%R; color label magenta;select 110 and *.ca; label %m%R; color label magenta;select 111 and *.ca; label %m%R; color label magenta;select 112 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">2</td><td valign="middle" align="center">0.12</td> <td valign="middle" align="center">68</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=1nvw target="_blank">1nvwQ</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=GNP target="_blank">GNP</a> </td>  <td valign="middle" align=center><a href="CH_CF_1nvwQ_BS02_GNP.pdb" download>Rep</a>, <a href="CH_complex2.pdb" download>Mult</a></td>  <td valign="middle" align="left">25,26,27,80,81,83,84,110,111,112</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_3rslA_BS09_RSF.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 3,25,62,65; color blue; wireframe 75; spacefill 100;select 3 and *.ca; label %m%R; color label magenta;select 25 and *.ca; label %m%R; color label magenta;select 62 and *.ca; label %m%R; color label magenta;select 65 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">3</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">18</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3rsl target="_blank">3rslA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=RSF target="_blank">RSF</a> </td>  <td valign="middle" align=center><a href="CH_TM_3rslA_BS09_RSF.pdb" download>Rep</a>, <a href="CH_complex3.pdb" download>Mult</a></td>  <td valign="middle" align="left">3,25,62,65</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_3nkvA_BS03_AMP.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 3,5,20,22,37; color blue; wireframe 75; spacefill 100;select 3 and *.ca; label %m%R; color label magenta;select 5 and *.ca; label %m%R; color label magenta;select 20 and *.ca; label %m%R; color label magenta;select 22 and *.ca; label %m%R; color label magenta;select 37 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">4</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">15</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3nkv target="_blank">3nkvA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=AMP target="_blank">AMP</a> </td>  <td valign="middle" align=center><a href="CH_TM_3nkvA_BS03_AMP.pdb" download>Rep</a>, <a href="CH_complex4.pdb" download>Mult</a></td>  <td valign="middle" align="left">3,5,20,22,37</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_3v4fA_BS03_DTU.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 34,58,61,62,65; color blue; wireframe 75; spacefill 100;select 34 and *.ca; label %m%R; color label magenta;select 58 and *.ca; label %m%R; color label magenta;select 61 and *.ca; label %m%R; color label magenta;select 62 and *.ca; label %m%R; color label magenta;select 65 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">5</td><td valign="middle" align="center">0.01</td> <td valign="middle" align="center">4</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=3v4f target="_blank">3v4fA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=DTU target="_blank">DTU</a> </td>  <td valign="middle" align=center><a href="CH_CF_3v4fA_BS03_DTU.pdb" download>Rep</a>, <a href="CH_complex5.pdb" download>Mult</a></td>  <td valign="middle" align="left">34,58,61,62,65</td>    </tr>
       </table>
       <br/>
       <table  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
       <tr><td valign="top"></td><td align="justify"><a href=prob.txt target="_blank">Download</a> the residue-specific ligand binding probability, which is estimated by SVM. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.inf target="_blank">Download</a> the all possible binding ligands and detailed prediction summary. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.clr target="_blank">Download</a> the templates clustering results. </td></tr> 
        <tr><td valign="top">(a)</td><td align="justify"><b>C-score</b> is the confidence score of the prediction. C-score ranges [0-1], where a higher score indicates a more reliable prediction. </td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><b>Cluster size</b> is the total number of templates in a cluster. </td></tr>
       <tr><td valign="top">(c)</td><td align="justify"><b>Lig Name</b> is name of possible binding ligand. Click the name to view its information in <a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/>the BioLiP database</a>.</td></tr>
       <tr><td valign="top">(d)</td><td align="justify"><b>Rep</b> is a single complex structure with the most representative ligand in the cluster, i.e., the one listed in the <b>Lig Name</b> column. <br>
 <b>Mult</b> is the complex structures with all potential binding ligands in the cluster.</td></tr>
<br>       </table></td></tr>
     </table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:430px;">
   <table><tr><td style= "valign:middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Enzyme Commission (EC) numbers and active sites</font>
   </td></tr></table>
   </div>
   <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; cartoons;      set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top">
             <table  border="0" style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"> <th>Click<br>to view</th><th >Rank</th><th >Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov</th><th >EC Number</th><th align=left>Active Site Residues</th></tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.334</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2f9m" target="_blank">2f9mA</a></td><td align="center"><font face="Arial">0.788</font></td><td align="center"><font face="Arial"> 1.98</font></td><td align="center"><font face="Arial">0.357</font></td><td align="right"><font face="Arial">0.853</font></td>
         <td align="left"><span title="Small monomeric GTPase."><a href="http://ca.expasy.org/enzyme/3.6.5.2" target="_blank"><font face="Arial">3.6.5.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 79; wireframe 70; spacefill 100; color cpk;  select 79 and *.ca; label %m%R; color label magenta; select 96; wireframe 70; spacefill 100; color cpk;  select 96 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.333</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2bme" target="_blank">2bmeA</a></td><td align="center"><font face="Arial">0.791</font></td><td align="center"><font face="Arial"> 1.64</font></td><td align="center"><font face="Arial">0.412</font></td><td align="right"><font face="Arial">0.834</font></td>
         <td align="left"><span title="Small monomeric GTPase."><a href="http://ca.expasy.org/enzyme/3.6.5.2" target="_blank"><font face="Arial">3.6.5.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">79,96</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.330</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bfk" target="_blank">3bfkB</a></td><td align="center"><font face="Arial">0.774</font></td><td align="center"><font face="Arial"> 1.99</font></td><td align="center"><font face="Arial">0.345</font></td><td align="right"><font face="Arial">0.847</font></td>
         <td align="left"><span title="Small monomeric GTPase."><a href="http://ca.expasy.org/enzyme/3.6.5.2" target="_blank"><font face="Arial">3.6.5.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.326</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bc1" target="_blank">3bc1A</a></td><td align="center"><font face="Arial">0.776</font></td><td align="center"><font face="Arial"> 1.36</font></td><td align="center"><font face="Arial">0.370</font></td><td align="right"><font face="Arial">0.828</font></td>
         <td align="left"><span title="Small monomeric GTPase."><a href="http://ca.expasy.org/enzyme/3.6.5.2" target="_blank"><font face="Arial">3.6.5.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.316</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2atx" target="_blank">2atxA</a></td><td align="center"><font face="Arial">0.745</font></td><td align="center"><font face="Arial"> 1.61</font></td><td align="center"><font face="Arial">0.260</font></td><td align="right"><font face="Arial">0.804</font></td>
         <td align="left"><span title="Small monomeric GTPase."><a href="http://ca.expasy.org/enzyme/3.6.5.2" target="_blank"><font face="Arial">3.6.5.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         </table><br/>
         <table border="0"  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
          <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the EC number prediction. Cscore<sup>EC</sup> values range in between [0-1]; <br>where a higher score indicates a more reliable EC number prediction.</td></tr>
           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
           <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided <br>by length of the query protein.</td></tr>
         </table>
        </td></tr></table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Gene Ontology (GO) terms</font>
      </td></tr></table>
   </div>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr background:#FF9900;><th colspan=8 align="left"><font face="Arial" size="3">Top 10 homologous GO templates in PDB</font><th><td>&nbsp;</td></tr>
  <tr bgcolor="#DEDEDE"><th ><font face="Arial">Rank</font></th><th><font face="Arial">Cscore<sup>GO</sup></font></th><th><font face="Arial">TM-score</font></th><th><font face="Arial">RMSD<sup>a</sup></font></th><th><font face="Arial">IDEN<sup>a</sup></font></th><th><font face="Arial">Cov</font></th><th align="left"><font face="Arial">PDB Hit</font></th><th align="left"><font face="Arial">Associated GO Terms</font></th></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">1</font></td><td align="center"  valign="middle"><font face="Arial"> 0.48</font></td><td  valign="middle" align="center"><font face="Arial">0.9129</font></td><td align="center"  valign="middle"><font face="Arial"> 1.66</font></td><td align="center"  valign="middle"><font face="Arial"> 0.34</font></td><td  valign="middle" align="center"><font face="Arial"> 0.97</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ukv" target="_blank">1ukvY</a></td><td valign="middle"><font face="Arial">
<span title="SNARE binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000149 target="_blank">GO:0000149</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="retrograde vesicle-mediated transport, Golgi to ER"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006890 target="_blank">GO:0006890</a></span>
<span title="Golgi membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000139 target="_blank">GO:0000139</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="ER to Golgi vesicle-mediated transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006888 target="_blank">GO:0006888</a></span>
<span title="Golgi apparatus"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005794 target="_blank">GO:0005794</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
<span title="trans-Golgi network"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005802 target="_blank">GO:0005802</a></span>
<span title="endoplasmic reticulum membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005789 target="_blank">GO:0005789</a></span>
<span title="protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515 target="_blank">GO:0005515</a></span>
<span title="early endosome to Golgi transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0034498 target="_blank">GO:0034498</a></span>
<span title="cytoplasm"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005737 target="_blank">GO:0005737</a></span>
<span title="CVT pathway"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032258 target="_blank">GO:0032258</a></span>
<span title="pre-autophagosomal structure"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000407 target="_blank">GO:0000407</a></span>
<span title="endocytic recycling"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032456 target="_blank">GO:0032456</a></span>
<span title="Golgi stack"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005795 target="_blank">GO:0005795</a></span>
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="Golgi vesicle budding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0048194 target="_blank">GO:0048194</a></span>
<span title="endoplasmic reticulum"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005783 target="_blank">GO:0005783</a></span>
<span title="vesicle-mediated transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016192 target="_blank">GO:0016192</a></span>
<span title="macroautophagy"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016236 target="_blank">GO:0016236</a></span>
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="mitochondrion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739 target="_blank">GO:0005739</a></span>
<span title="Golgi vesicle docking"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0048211 target="_blank">GO:0048211</a></span>
<span title="COPII-coated vesicle budding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0090114 target="_blank">GO:0090114</a></span>
<span title="GTP catabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006184 target="_blank">GO:0006184</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="GTPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003924 target="_blank">GO:0003924</a></span>
<span title="protein complex assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006461 target="_blank">GO:0006461</a></span>
<span title="SNARE complex disassembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0035494 target="_blank">GO:0035494</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">2</font></td><td align="center"  valign="middle"><font face="Arial"> 0.44</font></td><td  valign="middle" align="center"><font face="Arial">0.7602</font></td><td align="center"  valign="middle"><font face="Arial"> 1.31</font></td><td align="center"  valign="middle"><font face="Arial"> 0.40</font></td><td  valign="middle" align="center"><font face="Arial"> 0.80</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1n6o" target="_blank">1n6oA</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">3</font></td><td align="center"  valign="middle"><font face="Arial"> 0.44</font></td><td  valign="middle" align="center"><font face="Arial">0.7915</font></td><td align="center"  valign="middle"><font face="Arial"> 1.64</font></td><td align="center"  valign="middle"><font face="Arial"> 0.41</font></td><td  valign="middle" align="center"><font face="Arial"> 0.83</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2bme" target="_blank">2bmeA</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">4</font></td><td align="center"  valign="middle"><font face="Arial"> 0.44</font></td><td  valign="middle" align="center"><font face="Arial">0.7345</font></td><td align="center"  valign="middle"><font face="Arial"> 1.41</font></td><td align="center"  valign="middle"><font face="Arial"> 0.41</font></td><td  valign="middle" align="center"><font face="Arial"> 0.77</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2e9s" target="_blank">2e9sC</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">5</font></td><td align="center"  valign="middle"><font face="Arial"> 0.43</font></td><td  valign="middle" align="center"><font face="Arial">0.7874</font></td><td align="center"  valign="middle"><font face="Arial"> 1.79</font></td><td align="center"  valign="middle"><font face="Arial"> 0.42</font></td><td  valign="middle" align="center"><font face="Arial"> 0.85</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2o52" target="_blank">2o52A</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">6</font></td><td align="center"  valign="middle"><font face="Arial"> 0.43</font></td><td  valign="middle" align="center"><font face="Arial">0.7522</font></td><td align="center"  valign="middle"><font face="Arial"> 1.43</font></td><td align="center"  valign="middle"><font face="Arial"> 0.37</font></td><td  valign="middle" align="center"><font face="Arial"> 0.80</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1z08" target="_blank">1z08C</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">7</font></td><td align="center"  valign="middle"><font face="Arial"> 0.42</font></td><td  valign="middle" align="center"><font face="Arial">0.7823</font></td><td align="center"  valign="middle"><font face="Arial"> 1.74</font></td><td align="center"  valign="middle"><font face="Arial"> 0.41</font></td><td  valign="middle" align="center"><font face="Arial"> 0.83</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2a5j" target="_blank">2a5jA</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">8</font></td><td align="center"  valign="middle"><font face="Arial"> 0.42</font></td><td  valign="middle" align="center"><font face="Arial">0.7714</font></td><td align="center"  valign="middle"><font face="Arial"> 1.16</font></td><td align="center"  valign="middle"><font face="Arial"> 0.38</font></td><td  valign="middle" align="center"><font face="Arial"> 0.81</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2il1" target="_blank">2il1A</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">9</font></td><td align="center"  valign="middle"><font face="Arial"> 0.41</font></td><td  valign="middle" align="center"><font face="Arial">0.7742</font></td><td align="center"  valign="middle"><font face="Arial"> 1.99</font></td><td align="center"  valign="middle"><font face="Arial"> 0.34</font></td><td  valign="middle" align="center"><font face="Arial"> 0.85</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3bfk" target="_blank">3bfkB</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">10</font></td><td align="center"  valign="middle"><font face="Arial"> 0.40</font></td><td  valign="middle" align="center"><font face="Arial">0.7883</font></td><td align="center"  valign="middle"><font face="Arial"> 1.98</font></td><td align="center"  valign="middle"><font face="Arial"> 0.36</font></td><td  valign="middle" align="center"><font face="Arial"> 0.85</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2f9m" target="_blank">2f9mA</a></td><td valign="middle"><font face="Arial">
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span>
<span title="protein transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015031 target="_blank">GO:0015031</a></span>
<span title="GTP catabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006184 target="_blank">GO:0006184</a></span>
<span title="GDP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019003 target="_blank">GO:0019003</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="cell cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007049 target="_blank">GO:0007049</a></span>
<span title="GTPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003924 target="_blank">GO:0003924</a></span>
<span title="plasma membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005886 target="_blank">GO:0005886</a></span>
<span title="mitochondrion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739 target="_blank">GO:0005739</a></span>
<span title="cytoplasmic membrane-bounded vesicle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016023 target="_blank">GO:0016023</a></span>
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
</font></td></tr>
   </table>
<p>
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
     <tr ><th colspan=6 align="left"><font face="Arial" size="3"><br>Consensus prediction of GO terms</font><th><td>&nbsp;</td></tr>
      <tr><td>
      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Molecular Function</b></font></td>
      <td align="center"><font face="Arial"><span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span></font></td>
      <td align="center"><font face="Arial"><span title="GTPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003924 target="_blank">GO:0003924</a></span></font></td>
      <td align="center"><font face="Arial"><span title="SNARE binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000149 target="_blank">GO:0000149</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.95</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Biological Process</b></font></td>
      <td align="center"><font face="Arial"><span title="small GTPase mediated signal transduction"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007264 target="_blank">GO:0007264</a></span></font></td>
      <td align="center"><font face="Arial"><span title="GTP catabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006184 target="_blank">GO:0006184</a></span></font></td>
      <td align="center"><font face="Arial"><span title="Golgi vesicle docking"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0048211 target="_blank">GO:0048211</a></span></font></td>
      <td align="center"><font face="Arial"><span title="protein complex assembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006461 target="_blank">GO:0006461</a></span></font></td>
      <td align="center"><font face="Arial"><span title="COPII-coated vesicle budding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0090114 target="_blank">GO:0090114</a></span></font></td>
      <td align="center"><font face="Arial"><span title="CVT pathway"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032258 target="_blank">GO:0032258</a></span></font></td>
      <td align="center"><font face="Arial"><span title="Golgi vesicle budding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0048194 target="_blank">GO:0048194</a></span></font></td>
      <td align="center"><font face="Arial"><span title="macroautophagy"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016236 target="_blank">GO:0016236</a></span></font></td>
      <td align="center"><font face="Arial"><span title="SNARE complex disassembly"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0035494 target="_blank">GO:0035494</a></span></font></td>
      <td align="center"><font face="Arial"><span title="early endosome to Golgi transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0034498 target="_blank">GO:0034498</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.95</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Cellular Component</b></font></td>
      <td align="center"><font face="Arial"><span title="mitochondrion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005739 target="_blank">GO:0005739</a></span></font></td>
      <td align="center"><font face="Arial"><span title="Golgi stack"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005795 target="_blank">GO:0005795</a></span></font></td>
      <td align="center"><font face="Arial"><span title="trans-Golgi network"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005802 target="_blank">GO:0005802</a></span></font></td>
      <td align="center"><font face="Arial"><span title="pre-autophagosomal structure"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000407 target="_blank">GO:0000407</a></span></font></td>
      <td align="center"><font face="Arial"><span title="endoplasmic reticulum membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005789 target="_blank">GO:0005789</a></span></font></td>
      <td align="center"><font face="Arial"><span title="Golgi membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000139 target="_blank">GO:0000139</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
      <td align="center"><font face="Arial">0.48</font></td>
  </tr>
   </table>
  </td></tr></table>
<p>
       <table  style="width:1200px;font-size:12px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is a combined measure for evaluating global and local similarity between query and template protein. It's range is [0-1] and higher values indicate more confident predictions.</td></tr>
      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
      <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consensus GO terms amongst the top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template.</td></tr>
     </table>
  </div>
</td></tr>
<tr><td><br/></td></tr>
<tr><td>
<h3 align="center">[Click on <a href="S552663_results.tar.bz2" download>S552663_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<hr><br/>
<div style="position:relative;left:5px;"><table style="font-family:Arial;font-size:14px;"><tr><td colspan=2>Please cite the following articles when you use the I-TASSER server:</td><tr>
<tr><td valign="top">1.</td><td>Yang Zhang. I-TASSER: Fully automated protein structure prediction in CASP8. Proteins, 77 (Suppl 9): 100-113, 2009.</td></tr>
<tr><td valign="top">2.</td><td>Ambrish Roy, Jianyi Yang, Yang Zhang. COFACTOR: an accurate comparative algorithm for structure-based protein function annotation. Nucleic Acids Research, 40: W471-W477, 2012.</td></tr>
<tr><td valign="top">3.</td><td> Jianyi Yang, Yang Zhang. I-TASSER server: new development for protein structure and function predictions, Nucleic Acids Research, 43: W174-W181, 2015.</td></tr>
</table></div><br></td></tr>
</table></body>
</html>
