<html>
<head>
  <title>I-TASSER results</title>
</head>

<script type="text/javascript" src="../../bin/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="../../bin/jmol/Jmol2.js"></script>
<body bgcolor="#FFFFFF">
<font face="Times New Roman">
[<a href="http://zhanglab.ccmb.med.umich.edu">Home</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]
<br/>
<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">
<tr><td>
  <h1 align="center">I-TASSER results for job id S550642</h1>
<center> (Click on <a href="S550642_results.tar.bz2" download>S550642_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page. Click on <a href=http://zhanglab.ccmb.med.umich.edu/I-TASSER/annotation target=_blank>Annotation of I-TASSER Output</a> to read the instructions for how to interpret the results on this page. Model results are kept on the server for 60 days, there is no way to retrieve the modeling data older than 2 months)
</center><br>
</td></tr>
<tr><td>
   <div style="background:#6599FF;width:350px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence in <a href=http://zhanglab.ccmb.med.umich.edu/FASTA/ target="_blank">FASTA format</a></font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
   <tr><td colspan="2">>protein<br>
HQVRQPEDVIDTVYWSNEDYESFSIIDDKAFNNNEFTIGWYHSHPGFKVMMSQLDVKTTL<br/>
SYQQFNNLAVSLVFNPTRLIRQIEVPYKKGDPIKQLEEDPGFKIFRLDNINSGIEASYHE<br/>
VSYEIQGYESMEQLVRQTQKFIIEITNFFPKDNIFKTYEKYITTKINDLNSKLLGTEEYL<br/>
TTLERKGEGHRTKEVLENQVKDIRKFVAETFINIENIKEFMKYLEFKERHRVIPKVEEIL<br/>
NNWDEAVSKLNDKFTELANKF<br/>
</td></tr>
   </table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;<br/>HQVRQPEDVIDTVYWSNEDYESFSIIDDKAFNNNEFTIGWYHSHPGFKVMMSQLDVKTTLSYQQFNNLAVSLVFNPTRLIRQIEVPYKKGDPIKQLEEDPGFKIFRLDNINSGIEASYHEVSYEIQGYESMEQLVRQTQKFIIEITNFFPKDNIFKTYEKYITTKINDLNSKLLGTEEYLTTLERKGEGHRTKEVLENQVKDIRKFVAETFINIENIKEFMKYLEFKERHRVIPKVEEILNNWDEAVSKLNDKFTELANKF</td></tr>
    <tr><td><font="Arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">CCCCCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCC<font color="blue">SSSSSS</font>CCCCCCCCCC<font color="red">HHHHHHHHHHHHH</font>CCC<font color="blue">SSSSSS</font>CCCCCCCCC<font color="blue">SSSSS</font>CCCCCCCCCCCCCCCCCCCCCCCCCC<font color="red">HH</font>CC<font color="blue">SSS</font>CCC<font color="red">HH</font>CCCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCCC<font color="red">HHHHHH</font>CC<font color="red">HHHHHHHHHHHHHHHHH</font>CC</td>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font></td><td BGCOLOR="#ECECEC">988987888883540217999999999997298882577761799858788755888899886206883899989865566653552046887666678885312462002442003326410000025699999988799998886486256899899999999999999999999997632135655007899999988899999999854999999999987266578882077785057989888888999885349</td></tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC"><font color=red>H</font>:Helix; <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;<br/>HQVRQPEDVIDTVYWSNEDYESFSIIDDKAFNNNEFTIGWYHSHPGFKVMMSQLDVKTTLSYQQFNNLAVSLVFNPTRLIRQIEVPYKKGDPIKQLEEDPGFKIFRLDNINSGIEASYHEVSYEIQGYESMEQLVRQTQKFIIEITNFFPKDNIFKTYEKYITTKINDLNSKLLGTEEYLTTLERKGEGHRTKEVLENQVKDIRKFVAETFINIENIKEFMKYLEFKERHRVIPKVEEILNNWDEAVSKLNDKFTELANKF</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">862644653434444343420320242154163532000000022323020133114213313303430000000114324433021231332443467641432427414432333333031415324434213302442124114434344114423530352044025304513531463454456444453145235434422343244153024203430135435345424532442343133134303511767</td><tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>
   </td></tr></table>
   </div>
   <br/>

<font size=2>
(B-factor is a value to indicate the extent of the inherent thermal 

mobility of residues/atoms in proteins. In I-TASSER, this value is 

deduced from threading template proteins from the PDB in combination with the 

sequence profiles derived from sequence databases. The reported B-factor 

profile in the figure below corresponds to the normalized B-factor of the 

target protein, defined by B=(B'-u)/s, where B' is the raw B-factor 

value, u and s are respectively the mean and standard deviation of the raw 

B-factors along the sequence. 

<a href="lscore.txt" target="_blank"  >Click here to read more about predicted 

normalized B-factor</a>) 

</font> 

<br/> 

   <br/> 

   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr> <td align =center > <img src=BFP.png>  </td>  </tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 threading templates used by I-TASSER</font>
   </td></tr></table>
   </div>
   <br/>
   
<font size=2> 

(I-TASSER modeling starts from the structure templates identified by LOMETS from the PDB library. 

LOMETS is a meta-server threading approach containing multiple threading programs, 

where each threading program can generate tens of thousands of 

template alignments. I-TASSER only uses the templates of the highest significance in the threading alignments, 

the significance of which are measured by the Z-score, i.e. the difference between the raw and average scores 

in the unit of standard deviation. The templates in this section are the 10 best templates selected from 

the LOMETS threading programs. Usually, one template of the highest Z-score is selected from each threading program, where the threading programs are sorted by the average performance in the large-scale benchmark test experiments.) 

</font> 

   <br/> 

   <br/> 

   <div style="position:relative;left:30px;">
   <table style="font-size:13px;background:#F2F2F2;">
      <tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Z-score</font></th><th valign="top"><font face="Arial">Download<br/>Align.</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;100&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;120&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;140&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;160&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;180&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;200&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;220&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;240&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;260<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;</td></tr>
      <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">Sec.Str<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCCCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHH</font>CCCCC<font color="blue">SSSSSS</font>CCCCCCCCCC<font color="red">HHHHHHHHHHHHH</font>CCC<font color="blue">SSSSSS</font>CCCCCCCCC<font color="blue">SSSSS</font>CCCCCCCCCCCCCCCCCCCCCCCCCC<font color="red">HH</font>CC<font color="blue">SSS</font>CCC<font color="red">HH</font>CCCC<font color="red">HHHHHHHHHHHHHHHHH</font>CCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCC<font color="red">HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</font>CCCCCCCCC<font color="red">HHHHHH</font>CC<font color="red">HHHHHHHHHHHHHHHHH</font>CC<br/>HQVRQPEDVIDTVYWSNEDYESFSIIDDKAFNNNEFTIGWYHSHPGFKVMMSQLDVKTTLSYQQFNNLAVSLVFNPTRLIRQIEVPYKKGDPIKQLEEDPGFKIFRLDNINSGIEASYHEVSYEIQGYESMEQLVRQTQKFIIEITNFFPKDNIFKTYEKYITTKINDLNSKLLGTEEYLTTLERKGEGHRTKEVLENQVKDIRKFVAETFINIENIKEFMKYLEFKERHRVIPKVEEILNNWDEAVSKLNDKFTELANKF</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5a5t" target="_blank">5a5tH</a></td><td align ="center"><font face="Arial"> 0.13</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.99</font></td><td align="center"><font face="Arial"> 1.23</font></td><td align="center"><a href="threading1.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">FPFP<font color="#00DCDC">Q</font>HTEDDAD--FDEVQ<font color="#3232AA">Y</font>QMEMMRSLRHV<font color="#00DCDC">N</font>IDHLHV<font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font>Q<font color="#FA9600">S</font>TY-YGSFVTRALLDSQF<font color="#FA9600">S</font><font color="#3232AA">Y</font><font color="#00DCDC">Q</font>HAIEES<font color="#0F820F">V</font>V<font color="#0F820F">L</font>IYD<font color="#D69682">P</font>IKTAQGSLKA<font color="#3232AA">Y</font>RLTPKLMEVCKEKD<font color="#3232AA">F</font>SPEA<font color="#0F820F">L</font>KKA<font color="#00DCDC">N</font>ITF<font color="#E60A0A">E</font>HMFE<font color="#E60A0A">E</font><font color="#0F820F">V</font>PIV<font color="#0F820F">I</font>KNSHLINV<font color="#0F820F">L</font>MWELE<font color="#145AFF">K</font>ADKHELLSLASS<font color="#00DCDC">N</font>HLGKNLQLLMDRVDEMSQDIVKYNT<font color="#3232AA">Y</font>MRNTSKQQQQKHQYQQRRQ<font color="#00DCDC">Q</font>ENMQ<font color="#145AFF">R</font>QSRG<font color="#E60A0A">E</font>PPLPE<font color="#E60A0A">E</font>DLSKLF<font color="#145AFF">K</font>PPQPPA<font color="#145AFF">R</font>MDSLLIAGQ<font color="#0F820F">I</font>NTYCQNIKEFTAQNLGK<font color="#0F820F">L</font>FMAQ</td></tr>
       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5mpb" target="_blank">5mpbA</a></td><td align ="center"><font face="Arial"> 0.14</font></td><td><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.84</font></td><td align="center"><font face="Arial"> 2.43</font></td><td align="center"><a href="threading2.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-AMP-QSGTGVS<font color="#0F820F">V</font>EAVDDVFQAKMMDML<font color="#145AFF">K</font>QTGRDQMVV<font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font><font color="#3232AA">F</font>GCWL<font color="#FA9600">S</font>SV<font color="#E60A0A">D</font><font color="#0F820F">V</font>N<font color="#FA9600">T</font>QK<font color="#FA9600">S</font>FE<font color="#00DCDC">Q</font>L<font color="#00DCDC">N</font>SR<font color="#333333">A</font><font color="#0F820F">V</font>AV<font color="#0F820F">V</font>VD<font color="#D69682">P</font>IQSVK-GK<font color="#0F820F">V</font>VIDAFRL<font color="#0F820F">I</font>DTTSNTGLLNKANIQALIH<font color="#666666">G</font>LNRH--YY<font color="#FA9600">S</font>LN<font color="#0F820F">I</font>DYHKTAKETKMLMNLHKEQWQSGLKMYD-YEEK<font color="#E60A0A">E</font>ESNLAATKSMVKIAEQYSKRIEEEKELT<font color="#E60A0A">E</font>EELKTRPKKHLSETADETLENNI--VSVLTAGVNSVAI<font color="#145AFF">K</font>----------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5mpb" target="_blank">5mpbV</a></td><td align ="center"><font face="Arial"> 0.18</font></td><td><font face="Arial"> 0.26</font></td><td><font face="Arial"> 0.86</font></td><td align="center"><font face="Arial"> 1.04</font></td><td align="center"><a href="threading3.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">VFAMPQSGTGVS<font color="#0F820F">V</font>EAVDDVFQAKMMDML<font color="#145AFF">K</font>QTGRDQMVV<font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font><font color="#3232AA">F</font>GCWL<font color="#FA9600">S</font>SV<font color="#E60A0A">D</font><font color="#0F820F">V</font>N<font color="#FA9600">T</font>QK<font color="#FA9600">S</font>FE<font color="#00DCDC">Q</font>L<font color="#00DCDC">N</font>SR<font color="#333333">A</font><font color="#0F820F">V</font>AV<font color="#0F820F">V</font>VD<font color="#D69682">P</font>IQSGKVVIDAFRLI<font color="#E60A0A">D</font>TGEPRQTTSNTGLLNKA<font color="#00DCDC">N</font><font color="#0F820F">I</font>QAL<font color="#0F820F">I</font>HGLR<font color="#8282D2">H</font>YY<font color="#FA9600">S</font>LN<font color="#0F820F">I</font>DYHKTAKETKMLMNLHKEQWQSGLKMYD-YEEK<font color="#E60A0A">E</font>ESNLAATKSMVKIAEQYSKRI-----EE<font color="#E60A0A">E</font>KEL<font color="#FA9600">T</font>E<font color="#E60A0A">E</font>E<font color="#0F820F">L</font>KTRYV<font color="#E60A0A">D</font>PK<font color="#145AFF">K</font>HLS<font color="#E60A0A">E</font><font color="#FA9600">T</font>ATLEN<font color="#00DCDC">N</font><font color="#0F820F">I</font>VSVLTAGVNSVAIK------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10</a></td><td align ="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.75</font></td><td align="center"><font face="Arial"> 2.75</font></td><td align="center"><a href="threading4.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">PVEGTE--TRVNAQAAAYE<font color="#3232AA">Y</font>MAYIENAK-QVGRL<font color="#E60A0A">E</font>NA<font color="#0F820F">I</font><font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font>YGCWL<font color="#FA9600">S</font>GI<font color="#E60A0A">D</font><font color="#0F820F">V</font>S<font color="#FA9600">T</font>QMLN<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font><font color="#3232AA">F</font>QEPF<font color="#0F820F">V</font>AV<font color="#0F820F">V</font>ID<font color="#D69682">P</font><font color="#FA9600">T</font><font color="#145AFF">R</font>T<font color="#0F820F">I</font>SA---------------GKVNLGA<font color="#3232AA">F</font><font color="#145AFF">R</font>TYPKGY<font color="#666666">G</font>VCKQ<font color="#3232AA">Y</font>YALEVSYFKSSLDRK<font color="#0F820F">L</font>ELLWN<font color="#145AFF">K</font>YWVNTLSSSSLL----<font color="#FA9600">T</font>NAD<font color="#3232AA">Y</font>T<font color="#FA9600">T</font>GQVF<font color="#E60A0A">D</font><font color="#0F820F">L</font>SE<font color="#145AFF">K</font><font color="#0F820F">L</font>EQS<font color="#E60A0A">E</font>AQ<font color="#0F820F">L</font>GRGSEDKLAK--------ATR<font color="#E60A0A">D</font>SC<font color="#145AFF">K</font>TTI<font color="#E60A0A">E</font>AIHGSQV<font color="#0F820F">I</font><font color="#145AFF">K</font>DKFNQINI-----------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10</a></td><td align ="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.83</font></td><td align="center"><font face="Arial"> 2.48</font></td><td align="center"><a href="threading5.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">DSLPGT<font color="#E60A0A">E</font>TRVNA-QAAAYE<font color="#3232AA">Y</font>MAAY<font color="#0F820F">I</font>ENA<font color="#145AFF">K</font>QVGRL<font color="#E60A0A">E</font>NA<font color="#0F820F">I</font><font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font>YGCWL<font color="#FA9600">S</font>GI<font color="#E60A0A">D</font><font color="#0F820F">V</font>S<font color="#FA9600">T</font>QMLN<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font><font color="#3232AA">F</font>QEPF<font color="#0F820F">V</font>AV<font color="#0F820F">V</font>ID<font color="#D69682">P</font><font color="#FA9600">T</font><font color="#145AFF">R</font>T<font color="#0F820F">I</font>SAGK<font color="#0F820F">V</font>NLGAFRTYPKGYGPSEYQTIP<font color="#0F820F">L</font>NK<font color="#0F820F">I</font>EDFGVHCKQYYAL<font color="#E60A0A">E</font>VSYFK<font color="#FA9600">S</font>SLDRKLLWN<font color="#145AFF">K</font>YWVNTLSSSS----LL<font color="#FA9600">T</font>NAD<font color="#3232AA">Y</font>T<font color="#FA9600">T</font>GQVF<font color="#E60A0A">D</font><font color="#0F820F">L</font>SE<font color="#145AFF">K</font><font color="#0F820F">L</font>EQS<font color="#E60A0A">E</font>AQ<font color="#0F820F">L</font>GRGS--------EDK<font color="#0F820F">L</font>AKATR<font color="#E60A0A">D</font>SC<font color="#145AFF">K</font>TTI<font color="#E60A0A">E</font>AIHGL--MSQVI<font color="#145AFF">K</font>DKL<font color="#3232AA">F</font>NQINI------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5a5t" target="_blank">5a5tH</a></td><td align ="center"><font face="Arial"> 0.13</font></td><td><font face="Arial"> 0.17</font></td><td><font face="Arial"> 0.98</font></td><td align="center"><font face="Arial"> 1.70</font></td><td align="center"><a href="threading6.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-----<font color="#D69682">P</font>QHTE<font color="#E60A0A">D</font>DADFDEVQ<font color="#3232AA">Y</font>QMEMMRSLRHV<font color="#00DCDC">N</font>IDHLHV<font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font>Q<font color="#FA9600">S</font>TY-YGSFVTRALLDSQF<font color="#FA9600">S</font><font color="#3232AA">Y</font><font color="#00DCDC">Q</font>HAIEES<font color="#0F820F">V</font>V<font color="#0F820F">L</font>IYD<font color="#D69682">P</font>IKTAQGSLKA<font color="#3232AA">Y</font>RLTPKLMEVCKEKD<font color="#3232AA">F</font>SPEA<font color="#0F820F">L</font>KKA<font color="#00DCDC">N</font>ITF<font color="#E60A0A">E</font>HMFE<font color="#E60A0A">E</font><font color="#0F820F">V</font>PIV<font color="#0F820F">I</font>KNSHLINV<font color="#0F820F">L</font>MWELE<font color="#145AFF">K</font>KSAVADKHEASS<font color="#00DCDC">N</font>HLGKNLQLLMDRVDEMSQDIVKYNT<font color="#3232AA">Y</font>MRNTSKQQQQKHQYQQRRQ<font color="#00DCDC">Q</font>ENMQ<font color="#145AFF">R</font>QSRG<font color="#E60A0A">E</font>PPLPE<font color="#E60A0A">E</font>DLSKLF<font color="#145AFF">K</font>PPQPPA<font color="#145AFF">R</font>MDSLLIAGQ<font color="#0F820F">I</font>NTYCQNIKEFTAQNLGK<font color="#0F820F">L</font>FMAQ</td></tr>
       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10</a></td><td align ="center"><font face="Arial"> 0.23</font></td><td><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.76</font></td><td align="center"><font face="Arial"> 3.51</font></td><td align="center"><a href="threading7.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--------------AAAYE<font color="#3232AA">Y</font>MAAY<font color="#0F820F">I</font>ENA<font color="#145AFF">K</font>QVGRL<font color="#E60A0A">E</font>NA<font color="#0F820F">I</font><font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font>YGCWL<font color="#FA9600">S</font>GI<font color="#E60A0A">D</font><font color="#0F820F">V</font>S<font color="#FA9600">T</font>QMLN<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font><font color="#3232AA">F</font>QEPF<font color="#0F820F">V</font>AV<font color="#0F820F">V</font>ID<font color="#D69682">P</font><font color="#FA9600">T</font><font color="#145AFF">R</font>T<font color="#0F820F">I</font>AGKVGAFRTYPKGYKPP<font color="#E60A0A">E</font>YQTIPLNKIEDFGV-HCKQ<font color="#3232AA">Y</font>YALEVSYFKSSLDKL<font color="#0F820F">L</font>ELLWN<font color="#145AFF">K</font>YWVNTLSSSS----LL<font color="#FA9600">T</font>NAD<font color="#3232AA">Y</font>T<font color="#FA9600">T</font>GQVF<font color="#E60A0A">D</font><font color="#0F820F">L</font>SE<font color="#145AFF">K</font><font color="#0F820F">L</font>EQS<font color="#E60A0A">E</font>AQ<font color="#0F820F">L</font>GRGS--------EDK<font color="#0F820F">L</font>AKATR<font color="#E60A0A">D</font>SC<font color="#145AFF">K</font>TTI<font color="#E60A0A">E</font>AIHGL--MSQVI<font color="#145AFF">K</font>DKL<font color="#3232AA">F</font>NQ---------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10E</a></td><td align ="center"><font face="Arial"> 0.24</font></td><td><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.84</font></td><td align="center"><font face="Arial"> 1.83</font></td><td align="center"><a href="threading8.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">LP<font color="#0F820F">V</font>EGT<font color="#E60A0A">E</font>TRVN-AQAAAYE<font color="#3232AA">Y</font>MAAY<font color="#0F820F">I</font>ENA<font color="#145AFF">K</font>QVGRL<font color="#E60A0A">E</font>NA<font color="#0F820F">I</font><font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font>YGCWL<font color="#FA9600">S</font>GI<font color="#E60A0A">D</font><font color="#0F820F">V</font>S<font color="#FA9600">T</font>QMLN<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font><font color="#3232AA">F</font>QEPF<font color="#0F820F">V</font>AV<font color="#0F820F">V</font>ID<font color="#D69682">P</font><font color="#FA9600">T</font><font color="#145AFF">R</font>T<font color="#0F820F">I</font>SAGKRT<font color="#3232AA">Y</font>P<font color="#145AFF">K</font><font color="#666666">G</font>YK-PPD<font color="#E60A0A">E</font>GPSEYQTIP<font color="#0F820F">L</font>NK<font color="#0F820F">I</font>EDFGVHCKQYYAL<font color="#E60A0A">E</font>VSYFK<font color="#FA9600">S</font>SLDRKLLELLWNKYWV<font color="#00DCDC">N</font>TLSS-SSLL<font color="#FA9600">T</font>NAD<font color="#3232AA">Y</font>T<font color="#FA9600">T</font>GQVF<font color="#E60A0A">D</font><font color="#0F820F">L</font>SE<font color="#145AFF">K</font><font color="#0F820F">L</font>EQS<font color="#E60A0A">E</font>AQ<font color="#0F820F">L</font>GRGSE--------DK<font color="#0F820F">L</font>AKATR<font color="#E60A0A">D</font>SC<font color="#145AFF">K</font>TTI<font color="#E60A0A">E</font>AIHGL--MSQVI<font color="#145AFF">K</font>DKL<font color="#3232AA">F</font>NQINI------------------------------</td></tr>
       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3ecm" target="_blank">3ecmA</a></td><td align ="center"><font face="Arial"> 0.20</font></td><td><font face="Arial"> 0.23</font></td><td><font face="Arial"> 0.91</font></td><td align="center"><font face="Arial"> 0.97</font></td><td align="center"><a href="threading9.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">DD<font color="#0F820F">V</font>PEN<font color="#E60A0A">E</font>EYW<font color="#E60A0A">D</font>LI<font color="#3232AA">Y</font>LGLKMFAR<font color="#3232AA">F</font>G<font color="#0F820F">I</font>CE--FLHCS<font color="#E60A0A">E</font>S<font color="#FA9600">T</font>LS<font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font>SNPYHNSTHSA<font color="#E60A0A">D</font><font color="#0F820F">V</font>LHATA<font color="#3232AA">Y</font>ERIKE<font color="#0F820F">L</font>IAATIHDVDHPG<font color="#145AFF">R</font>TNSFLCNA<font color="#666666">G</font>SELAI<font color="#0F820F">L</font>YN<font color="#E60A0A">D</font>TAV-----------L<font color="#E60A0A">E</font>SHHAALAFQLT<font color="#666666">G</font>FKN<font color="#FFCC00">M</font><font color="#E60A0A">E</font>RNY<font color="#145AFF">R</font>TLRQG<font color="#0F820F">I</font><font color="#0F820F">I</font>DMVLATEMTKH<font color="#3232AA">F</font>EHVN<font color="#145AFF">K</font>FVNS-----------INKP<font color="#0F820F">L</font>A<font color="#FA9600">T</font><font color="#0F820F">L</font><font color="#E60A0A">E</font>EN<font color="#666666">G</font><font color="#E60A0A">E</font>TDKNQ<font color="#E60A0A">E</font><font color="#0F820F">V</font>INTMLREN<font color="#145AFF">R</font>TLIKRML<font color="#0F820F">I</font>KLQYCI<font color="#E60A0A">E</font>WAARIS<font color="#3232AA">F</font>DRNTCS<font color="#0F820F">I</font><font color="#D69682">P</font><font color="#145AFF">K</font>ITDMFDA<font color="#B45AB4">W</font><font color="#E60A0A">D</font>DLMQH<font color="#0F820F">L</font>DNN<font color="#3232AA">F</font>KYWKDEM</td></tr>
       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10E</a></td><td align ="center"><font face="Arial"> 0.18</font></td><td><font face="Arial"> 0.25</font></td><td><font face="Arial"> 0.85</font></td><td align="center"><font face="Arial"> 4.64</font></td><td align="center"><a href="threading10.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-FALPV<font color="#E60A0A">E</font>GTETR<font color="#0F820F">V</font>NAAAYE<font color="#3232AA">Y</font>MAAY<font color="#0F820F">I</font>ENA<font color="#145AFF">K</font>QVGRL<font color="#E60A0A">E</font>NA<font color="#0F820F">I</font><font color="#666666">G</font><font color="#B45AB4">W</font><font color="#3232AA">Y</font><font color="#8282D2">H</font><font color="#FA9600">S</font><font color="#8282D2">H</font><font color="#D69682">P</font><font color="#666666">G</font>YGCWL<font color="#FA9600">S</font>GI<font color="#E60A0A">D</font><font color="#0F820F">V</font>S<font color="#FA9600">T</font>QMLN<font color="#00DCDC">Q</font><font color="#00DCDC">Q</font><font color="#3232AA">F</font>QEPF<font color="#0F820F">V</font>AV<font color="#0F820F">V</font>ID<font color="#D69682">P</font><font color="#FA9600">T</font><font color="#145AFF">R</font>T<font color="#0F820F">I</font>SAGK<font color="#0F820F">V</font>NLGAFRTYP-----------------------------------KGYKPPDEGPSEYQT<font color="#0F820F">I</font>PLNKIE<font color="#E60A0A">D</font>FGVHCKQ-<font color="#3232AA">Y</font>YALEVSYFK<font color="#FA9600">S</font>S<font color="#0F820F">L</font>DRKLLE<font color="#0F820F">L</font>LWNKYWVNTLSSSSL<font color="#0F820F">L</font>T<font color="#00DCDC">N</font>ADYTTGQVFDLSEKLEQSQLGRGSEDKLAKAT<font color="#145AFF">R</font>DSC<font color="#145AFF">K</font>TTIEAIHGLMSQVIKDKL<font color="#3232AA">F</font>NQIN---</td></tr>
      <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score >1 mean a good alignment and vice versa.</td></tr>
        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>
        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>
        <tr><td>&nbsp;</td><td align ="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
1: MUSTER&nbsp;&nbsp;&nbsp;2: FFAS-3D&nbsp;&nbsp;&nbsp;3: SPARKS-X&nbsp;&nbsp;&nbsp;4: HHSEARCH2&nbsp;&nbsp;&nbsp;5: HHSEARCH I&nbsp;&nbsp;&nbsp;6: Neff-PPAS&nbsp;&nbsp;&nbsp;7: HHSEARCH&nbsp;&nbsp;&nbsp;8: pGenTHREADER&nbsp;&nbsp;&nbsp;9: PROSPECT2&nbsp;&nbsp;&nbsp;10: PRC&nbsp;&nbsp;&nbsp;        </td></tr>
      </table>
      </td></tr></table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:380px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Top 5 final models predicted by I-TASSER </font>
   </td></tr></table>
   </div>
   <br/>
  
<font size=2>

(For each target, I-TASSER simulations generate a large ensemble of
structural conformations, called decoys. To select the final models, I-TASSER uses the SPICKER program to cluster
all the decoys based on the pair-wise structure similarity, and reports up to five models which
corresponds to the five largest structure clusters. The confidence of each model is quantitatively
measured by C-score that is calculated based on the significance of threading template alignments
and the convergence parameters of the structure assembly simulations.
C-score is typically in the range of [-5, 2], where a C-score of a higher value signifies a model with a higher 
confidence and vice-versa. TM-score and RMSD are estimated based on C-score and protein length following 
the correlation observed between these qualities. Since the top 5 models are ranked by the cluster size,
it is possible that the lower-rank models have a higher C-score in 
rare cases. Although the first model has a better
quality in most cases, it is also possible that the lower-rank models have a better quality than the
higher-rank models as seen in our benchmark tests. If the I-TASSER simulations converge, it
is possible to have less than 5 clusters generated; this is usually an indication that the
models have a good quality because of the converged simulations.)
<ul>
<li><a href=cscore.txt target="_blank">More about C-score</a>
<li><a href=lscore.html>Local structure accuracy profile of the top five models</a>
</ul>
</font>
<ul>   <div style="position:relative;left:1px;">
       <table  style="table-layout:fixed;width:1780px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
  <tr> <td colspan=5> <font size=2>(By right-click on the images, you can export image file or change the configurations, e.g. modifying the background color or stopping the spin of
your models)</font></td></tr>
       <tr>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model2.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model3.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model4.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model5.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin off;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       </tr> <tr> <td  align =left>
<li><a href="model1.pdb" download>Download Model 1</a>

<li>C-score=-2.50 (<a href="cscore.txt" target="_blank" >Read more about C-score</a>) 

<li>Estimated TM-score = 0.42&plusmn;0.14

<li>Estimated RMSD = 11.7&plusmn;4.5&Aring;

<br><br>

</td>
 <td  align =left>
<li><a href="model2.pdb" download>Download Model 2</a>

<li>C-score = -2.66 <br><br>

</td>
 <td  align =left>
<li><a href="model3.pdb" download>Download Model 3</a>

<li>C-score = -2.11 <br><br>

</td>
 <td  align =left>
<li><a href="model4.pdb" download>Download Model 4</a>

<li>C-score = -3.50 <br><br>

</td>
 <td  align =left>
<li><a href="model5.pdb" download>Download Model 5</a>

<li>C-score = -3.65 <br><br>

</td>
       </tr>
       </table>
    </div>
    </div>
  </td></tr>
  </ul>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:670px;">
      <table>
         <tr><td style= "valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins structurally close to the target in the PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>
      </table>
   </div>
   <br/>
   
<font size=2>

(After the structure assembly simulation, I-TASSER uses the TM-align structural alignment program to match the first I-TASSER model to all structures in the PDB library. This section reports the top 10 proteins from the PDB that have the closest structural similarity, 

i.e. the highest <a href=http://zhanglab.ccmb.med.umich.edu/TM-score>TM-score</a>, to the predicted I-TASSER model. Due to the structural similarity, these proteins often have similar function to the target. However, users are encouraged to use the data in the next section 'Predicted function using COACH' to infer the function of the target protein, since COACH has been extensively trained to derive biological functions from multi-source of sequence and structure features which has on average a higher accuracy than the function annotations derived only from the global structure comparison.)</font> 

   <br/>

   <br/>

   <div>
   <div style="position:relative;left:30px;background:#F2F2F2;">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
           <tr>
           <td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1_4d10E.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br/><br/>
                 <table border="0" style="font-family:Arial;font-size:13px;">
                    <tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th >Rank</th><th >PDB Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th>Cov</th><th>Alignment</th></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4d10E.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script></td><font face="Monospace">
                       <td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4d10" target="_blank">4d10E</a></font></td><td align="center">0.769</td><td valign="middle" align="center"> 2.04</td><td valign="middle" align="center">0.205</td><td valign="middle">0.843</td><td align="center"><a href="model1_4d10E.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5mpbV.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5mpb" target="_blank">5mpbV</a></font></td><td align="center">0.490</td><td valign="middle" align="center"> 3.61</td><td valign="middle" align="center">0.174</td><td valign="middle">0.598</td><td align="center"><a href="model1_5mpbV.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5l4kV.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5l4k" target="_blank">5l4kV</a></font></td><td align="center">0.482</td><td valign="middle" align="center"> 3.49</td><td valign="middle" align="center">0.133</td><td valign="middle">0.586</td><td align="center"><a href="model1_5l4kV.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4zktB.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4zkt" target="_blank">4zktB</a></font></td><td align="center">0.475</td><td valign="middle" align="center"> 6.07</td><td valign="middle" align="center">0.030</td><td valign="middle">0.801</td><td align="center"><a href="model1_4zktB.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3vuoA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3vuo" target="_blank">3vuoA</a></font></td><td align="center">0.473</td><td valign="middle" align="center"> 6.22</td><td valign="middle" align="center">0.030</td><td valign="middle">0.805</td><td align="center"><a href="model1_3vuoA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5a5tH.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5a5t" target="_blank">5a5tH</a></font></td><td align="center">0.471</td><td valign="middle" align="center"> 4.73</td><td valign="middle" align="center">0.103</td><td valign="middle">0.659</td><td align="center"><a href="model1_5a5tH.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5gjqV.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5gjq" target="_blank">5gjqV</a></font></td><td align="center">0.470</td><td valign="middle" align="center"> 3.52</td><td valign="middle" align="center">0.164</td><td valign="middle">0.559</td><td align="center"><a href="model1_5gjqV.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1jqoA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1jqo" target="_blank">1jqoA</a></font></td><td align="center">0.457</td><td valign="middle" align="center"> 5.91</td><td valign="middle" align="center">0.056</td><td valign="middle">0.770</td><td align="center"><a href="model1_1jqoA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_3v0aB.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3v0a" target="_blank">3v0aB</a></font></td><td align="center">0.444</td><td valign="middle" align="center"> 6.63</td><td valign="middle" align="center">0.049</td><td valign="middle">0.812</td><td align="center"><a href="model1_3v0aB.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_5n0bA1.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=5n0b" target="_blank">5n0bA1</a></font></td><td align="center">0.444</td><td valign="middle" align="center"> 6.12</td><td valign="middle" align="center">0.039</td><td valign="middle">0.751</td><td align="center"><a href="model1_5n0bA1.pdb" download><font face="Arial">Download</font></a></td></tr>
               </table><br/>
               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
                <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
                <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
               </table>
          </td></tr>
       </table>
    </div>
    </div>
  </td></tr>
  <tr><td>
  <br/><br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted function using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a> and <a href="http://zhanglab.ccmb.med.umich.edu/COACH" target="_blank">COACH</a></font>
   </td></tr></table>
   </div>
  <br/>
  
<font size=2>

(This section reports biological annotations of the target 
protein by COFACTOR and COACH based on the I-TASSER structure prediction. 
While COFACTOR deduces protein functions (ligand-binding sites, EC and GO)
using structure comparison and protein-protein networks,
COACH is a meta-server approach that combines multiple function
annotation results (on ligand-binding sites) from the COFACTOR, 
TM-SITE and S-SITE programs.)

</font>

<br/>

<br/>

  <tr><td>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Ligand binding sites</font>
      </td></tr></table>
   </div>
  <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td valign="top" align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load CH_TM_4k1rA_BS02_ZN.pdb; calculate structure rama; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 42,44,55; wireframe 75; spacefill 100; color blue;select 42 and *.ca; label %m%R; color label magenta;select 44 and *.ca; label %m%R; color label magenta;select 55 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;     set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT> jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT>
</td>
       <td valign="top">       <table border="0"  style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>C-score</th><th>Cluster<br>size</th> <th>PDB<br>Hit</th> <th>Lig<br>Name</th> <th>Download<br>Complex</th> <th align=left>Ligand Binding Site Residues</th> </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_4k1rA_BS02_ZN.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 42,44,55; color blue; wireframe 75; spacefill 100;select 42 and *.ca; label %m%R; color label magenta;select 44 and *.ca; label %m%R; color label magenta;select 55 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", true);</script></td>
    <font face="Monospace"><td valign="middle" align="center">1</td><td valign="middle" align="center">0.45</td> <td valign="middle" align="center">16</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=4k1r target="_blank">4k1rA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=ZN target="_blank">ZN</a> </td>  <td valign="middle" align=center><a href="CH_TM_4k1rA_BS02_ZN.pdb" download>Rep</a>, <a href="CH_complex1.pdb" download>Mult</a></td>  <td valign="middle" align="left">42,44,55</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_4uolC_BS01_GLY.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 56,58,60; color blue; wireframe 75; spacefill 100;select 56 and *.ca; label %m%R; color label magenta;select 58 and *.ca; label %m%R; color label magenta;select 60 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">2</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">1</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=4uol target="_blank">4uolC</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=GLY target="_blank">GLY</a> </td>  <td valign="middle" align=center><a href="CH_CF_4uolC_BS01_GLY.pdb" download>Rep</a>, <a href="CH_complex2.pdb" download>Mult</a></td>  <td valign="middle" align="left">56,58,60</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 1,3,14,21,35,37,38,39,40,41,62; color blue; wireframe 75; spacefill 100;select 1 and *.ca; label %m%R; color label magenta;select 3 and *.ca; label %m%R; color label magenta;select 14 and *.ca; label %m%R; color label magenta;select 21 and *.ca; label %m%R; color label magenta;select 35 and *.ca; label %m%R; color label magenta;select 37 and *.ca; label %m%R; color label magenta;select 38 and *.ca; label %m%R; color label magenta;select 39 and *.ca; label %m%R; color label magenta;select 40 and *.ca; label %m%R; color label magenta;select 41 and *.ca; label %m%R; color label magenta;select 62 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">3</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">1</td> <td valign="middle" align="left">N/A</td>  <td valign="middle" align="left"> N/A</td>  <td valign="middle" align="middle">N/A</td>    <td valign="middle" align="left">1,3,14,21,35,37,38,39,40,41,62</td>    </tr>
       </table>
       <br/>
       <table  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
       <tr><td valign="top"></td><td align="justify"><a href=prob.txt target="_blank">Download</a> the residue-specific ligand binding probability, which is estimated by SVM. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.inf target="_blank">Download</a> the all possible binding ligands and detailed prediction summary. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.clr target="_blank">Download</a> the templates clustering results. </td></tr> 
        <tr><td valign="top">(a)</td><td align="justify"><b>C-score</b> is the confidence score of the prediction. C-score ranges [0-1], where a higher score indicates a more reliable prediction. </td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><b>Cluster size</b> is the total number of templates in a cluster. </td></tr>
       <tr><td valign="top">(c)</td><td align="justify"><b>Lig Name</b> is name of possible binding ligand. Click the name to view its information in <a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/>the BioLiP database</a>.</td></tr>
       <tr><td valign="top">(d)</td><td align="justify"><b>Rep</b> is a single complex structure with the most representative ligand in the cluster, i.e., the one listed in the <b>Lig Name</b> column. <br>
 <b>Mult</b> is the complex structures with all potential binding ligands in the cluster.</td></tr>
<br>       </table></td></tr>
     </table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:430px;">
   <table><tr><td style= "valign:middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Enzyme Commission (EC) numbers and active sites</font>
   </td></tr></table>
   </div>
   <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; cartoons;      set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top">
             <table  border="0" style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"> <th>Click<br>to view</th><th >Rank</th><th >Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov</th><th >EC Number</th><th align=left>Active Site Residues</th></tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.089</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2eab" target="_blank">2eabA</a></td><td align="center"><font face="Arial">0.400</font></td><td align="center"><font face="Arial"> 5.75</font></td><td align="center"><font face="Arial">0.042</font></td><td align="right"><font face="Arial">0.644</font></td>
         <td align="left"><span title="1,2-alpha-L-fucosidase."><a href="http://ca.expasy.org/enzyme/3.2.1.63" target="_blank"><font face="Arial">3.2.1.63</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.089</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ps9" target="_blank">1ps9A</a></td><td align="center"><font face="Arial">0.395</font></td><td align="center"><font face="Arial"> 5.67</font></td><td align="center"><font face="Arial">0.038</font></td><td align="right"><font face="Arial">0.632</font></td>
         <td align="left"><span title="2,4-dienoyl-CoA reductase (NADPH)."><a href="http://ca.expasy.org/enzyme/1.3.1.34" target="_blank"><font face="Arial">1.3.1.34</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 48; wireframe 70; spacefill 100; color cpk;  select 48 and *.ca; label %m%R; color label magenta; select 202; wireframe 70; spacefill 100; color cpk;  select 202 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.089</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2pyw" target="_blank">2pywA</a></td><td align="center"><font face="Arial">0.396</font></td><td align="center"><font face="Arial"> 5.69</font></td><td align="center"><font face="Arial">0.087</font></td><td align="right"><font face="Arial">0.632</font></td>
         <td align="left"><span title="S-methyl-5-thioribose kinase."><a href="http://ca.expasy.org/enzyme/2.7.1.100" target="_blank"><font face="Arial">2.7.1.100</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">48,202</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.085</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2bmb" target="_blank">2bmbA</a></td><td align="center"><font face="Arial">0.399</font></td><td align="center"><font face="Arial"> 6.14</font></td><td align="center"><font face="Arial">0.049</font></td><td align="right"><font face="Arial">0.694</font></td>
         <td align="left"><span title="Dihydropteroate synthase."><a href="http://ca.expasy.org/enzyme/2.5.1.15" target="_blank"><font face="Arial">2.5.1.15</font></a></span>&nbsp;<span title="2-amino-4-hydroxy-6-hydroxymethyldihydropteridine diphosphokinase."><a href="http://ca.expasy.org/enzyme/2.7.6.3" target="_blank"><font face="Arial">2.7.6.3</font></a></span>&nbsp;<span title="Dihydroneopterin aldolase."><a href="http://ca.expasy.org/enzyme/4.1.2.25" target="_blank"><font face="Arial">4.1.2.25</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.082</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1jqo" target="_blank">1jqoA</a></td><td align="center"><font face="Arial">0.457</font></td><td align="center"><font face="Arial"> 5.91</font></td><td align="center"><font face="Arial">0.056</font></td><td align="right"><font face="Arial">0.770</font></td>
         <td align="left"><span title="Phosphoenolpyruvate carboxylase."><a href="http://ca.expasy.org/enzyme/4.1.1.31" target="_blank"><font face="Arial">4.1.1.31</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         </table><br/>
         <table border="0"  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
          <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the EC number prediction. Cscore<sup>EC</sup> values range in between [0-1]; <br>where a higher score indicates a more reliable EC number prediction.</td></tr>
           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
           <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided <br>by length of the query protein.</td></tr>
         </table>
        </td></tr></table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Gene Ontology (GO) terms</font>
      </td></tr></table>
   </div>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr background:#FF9900;><th colspan=8 align="left"><font face="Arial" size="3">Top 10 homologous GO templates in PDB</font><th><td>&nbsp;</td></tr>
  <tr bgcolor="#DEDEDE"><th ><font face="Arial">Rank</font></th><th><font face="Arial">Cscore<sup>GO</sup></font></th><th><font face="Arial">TM-score</font></th><th><font face="Arial">RMSD<sup>a</sup></font></th><th><font face="Arial">IDEN<sup>a</sup></font></th><th><font face="Arial">Cov</font></th><th align="left"><font face="Arial">PDB Hit</font></th><th align="left"><font face="Arial">Associated GO Terms</font></th></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">1</font></td><td align="center"  valign="middle"><font face="Arial"> 0.10</font></td><td  valign="middle" align="center"><font face="Arial">0.4108</font></td><td align="center"  valign="middle"><font face="Arial"> 4.85</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.60</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3o6x" target="_blank">3o6xA</a></td><td valign="middle"><font face="Arial">
<span title="catalytic activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003824 target="_blank">GO:0003824</a></span>
<span title="glutamate-ammonia ligase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004356 target="_blank">GO:0004356</a></span>
<span title="nitrogen compound metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006807 target="_blank">GO:0006807</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">2</font></td><td align="center"  valign="middle"><font face="Arial"> 0.10</font></td><td  valign="middle" align="center"><font face="Arial">0.4080</font></td><td align="center"  valign="middle"><font face="Arial"> 5.61</font></td><td align="center"  valign="middle"><font face="Arial"> 0.07</font></td><td  valign="middle" align="center"><font face="Arial"> 0.65</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3agr" target="_blank">3agrA</a></td><td valign="middle"><font face="Arial">
<span title="nucleoside-triphosphatase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0017111 target="_blank">GO:0017111</a></span>
<span title="hydrolase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016787 target="_blank">GO:0016787</a></span>
<span title="extracellular region"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005576 target="_blank">GO:0005576</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">3</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.4095</font></td><td align="center"  valign="middle"><font face="Arial"> 5.54</font></td><td align="center"  valign="middle"><font face="Arial"> 0.03</font></td><td  valign="middle" align="center"><font face="Arial"> 0.64</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2qen" target="_blank">2qenA</a></td><td valign="middle"><font face="Arial">
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">4</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.3932</font></td><td align="center"  valign="middle"><font face="Arial"> 6.24</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.66</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cb2" target="_blank">3cb2A</a></td><td valign="middle"><font face="Arial">
<span title="centrosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005813 target="_blank">GO:0005813</a></span>
<span title="microtubule cytoskeleton organization"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000226 target="_blank">GO:0000226</a></span>
<span title="polar microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005827 target="_blank">GO:0005827</a></span>
<span title="G2/M transition of mitotic cell cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000086 target="_blank">GO:0000086</a></span>
<span title="cell leading edge"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0031252 target="_blank">GO:0031252</a></span>
<span title="structural constituent of cytoskeleton"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005200 target="_blank">GO:0005200</a></span>
<span title="mitotic cell cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000278 target="_blank">GO:0000278</a></span>
<span title="protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515 target="_blank">GO:0005515</a></span>
<span title="pericentriolar material"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000242 target="_blank">GO:0000242</a></span>
<span title="cytoplasmic microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005881 target="_blank">GO:0005881</a></span>
<span title="gamma-tubulin complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000930 target="_blank">GO:0000930</a></span>
<span title="cytoskeleton"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005856 target="_blank">GO:0005856</a></span>
<span title="centriole"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005814 target="_blank">GO:0005814</a></span>
<span title="condensed nuclear chromosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000794 target="_blank">GO:0000794</a></span>
<span title="apical part of cell"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045177 target="_blank">GO:0045177</a></span>
<span title="meiotic spindle organization"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000212 target="_blank">GO:0000212</a></span>
<span title="spindle microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005876 target="_blank">GO:0005876</a></span>
<span title="microtubule organizing center"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005815 target="_blank">GO:0005815</a></span>
<span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span>
<span title="cytosol"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005829 target="_blank">GO:0005829</a></span>
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005874 target="_blank">GO:0005874</a></span>
<span title="cytoplasm"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005737 target="_blank">GO:0005737</a></span>
<span title="GTPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003924 target="_blank">GO:0003924</a></span>
<span title="GTP catabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006184 target="_blank">GO:0006184</a></span>
<span title="microtubule-based process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0007017 target="_blank">GO:0007017</a></span>
<span title="protein complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0043234 target="_blank">GO:0043234</a></span>
<span title="protein polymerization"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0051258 target="_blank">GO:0051258</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">5</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.4000</font></td><td align="center"  valign="middle"><font face="Arial"> 5.75</font></td><td align="center"  valign="middle"><font face="Arial"> 0.04</font></td><td  valign="middle" align="center"><font face="Arial"> 0.64</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2eab" target="_blank">2eabA</a></td><td valign="middle"><font face="Arial">
<span title="catalytic activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003824 target="_blank">GO:0003824</a></span>
<span title="alpha-L-fucosidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004560 target="_blank">GO:0004560</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">6</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.3990</font></td><td align="center"  valign="middle"><font face="Arial"> 6.14</font></td><td align="center"  valign="middle"><font face="Arial"> 0.05</font></td><td  valign="middle" align="center"><font face="Arial"> 0.69</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2bmb" target="_blank">2bmbA</a></td><td valign="middle"><font face="Arial">
<span title="2-amino-4-hydroxy-6-hydroxymethyldihydropteridine diphosphokinase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003848 target="_blank">GO:0003848</a></span>
<span title="folic acid-containing compound biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009396 target="_blank">GO:0009396</a></span>
<span title="pteridine-containing compound metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0042558 target="_blank">GO:0042558</a></span>
<span title="cellular metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0044237 target="_blank">GO:0044237</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">7</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.3952</font></td><td align="center"  valign="middle"><font face="Arial"> 5.67</font></td><td align="center"  valign="middle"><font face="Arial"> 0.04</font></td><td  valign="middle" align="center"><font face="Arial"> 0.63</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ps9" target="_blank">1ps9A</a></td><td valign="middle"><font face="Arial">
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="2,4-dienoyl-CoA reductase (NADPH) activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008670 target="_blank">GO:0008670</a></span>
<span title="dihydrodipicolinate synthase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008840 target="_blank">GO:0008840</a></span>
<span title="diaminopimelate biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019877 target="_blank">GO:0019877</a></span>
<span title="4 iron, 4 sulfur cluster binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0051539 target="_blank">GO:0051539</a></span>
<span title="oxidation-reduction process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0055114 target="_blank">GO:0055114</a></span>
<span title="oxidoreductase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016491 target="_blank">GO:0016491</a></span>
<span title="iron-sulfur cluster binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0051536 target="_blank">GO:0051536</a></span>
<span title="catalytic activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003824 target="_blank">GO:0003824</a></span>
<span title="metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008152 target="_blank">GO:0008152</a></span>
<span title="FMN binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0010181 target="_blank">GO:0010181</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">8</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial">0.4205</font></td><td align="center"  valign="middle"><font face="Arial"> 6.13</font></td><td align="center"  valign="middle"><font face="Arial"> 0.04</font></td><td  valign="middle" align="center"><font face="Arial"> 0.72</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3odm" target="_blank">3odmC</a></td><td valign="middle"><font face="Arial">
<span title="oxaloacetate metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006107 target="_blank">GO:0006107</a></span>
<span title="phosphoenolpyruvate carboxylase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008964 target="_blank">GO:0008964</a></span>
<span title="carbon fixation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015977 target="_blank">GO:0015977</a></span>
<span title="lyase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016829 target="_blank">GO:0016829</a></span>
<span title="catalytic activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003824 target="_blank">GO:0003824</a></span>
<span title="tricarboxylic acid cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006099 target="_blank">GO:0006099</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">9</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.3962</font></td><td align="center"  valign="middle"><font face="Arial"> 5.69</font></td><td align="center"  valign="middle"><font face="Arial"> 0.09</font></td><td  valign="middle" align="center"><font face="Arial"> 0.63</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2pyw" target="_blank">2pywA</a></td><td valign="middle"><font face="Arial">
<span title="cellular response to iron ion"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0071281 target="_blank">GO:0071281</a></span>
<span title="kinase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016301 target="_blank">GO:0016301</a></span>
<span title="methionine biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0009086 target="_blank">GO:0009086</a></span>
<span title="S-methyl-5-thioribose kinase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046522 target="_blank">GO:0046522</a></span>
<span title="cytosol"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005829 target="_blank">GO:0005829</a></span>
<span title="L-methionine salvage from methylthioadenosine"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019509 target="_blank">GO:0019509</a></span>
<span title="identical protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0042802 target="_blank">GO:0042802</a></span>
<span title="transferase activity, transferring phosphorus-containing groups"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016772 target="_blank">GO:0016772</a></span>
<span title="phosphorylation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016310 target="_blank">GO:0016310</a></span>
<span title="transferase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016740 target="_blank">GO:0016740</a></span>
<span title="cellular response to nitric oxide"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0071732 target="_blank">GO:0071732</a></span>
<span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span>
<span title="nucleotide binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000166 target="_blank">GO:0000166</a></span>
<span title="cellular amino acid biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008652 target="_blank">GO:0008652</a></span>
<span title="cellular response to ethylene stimulus"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0071369 target="_blank">GO:0071369</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">10</font></td><td align="center"  valign="middle"><font face="Arial"> 0.08</font></td><td  valign="middle" align="center"><font face="Arial">0.4573</font></td><td align="center"  valign="middle"><font face="Arial"> 5.91</font></td><td align="center"  valign="middle"><font face="Arial"> 0.06</font></td><td  valign="middle" align="center"><font face="Arial"> 0.77</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1jqo" target="_blank">1jqoA</a></td><td valign="middle"><font face="Arial">
<span title="metabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008152 target="_blank">GO:0008152</a></span>
<span title="catalytic activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003824 target="_blank">GO:0003824</a></span>
<span title="carbon fixation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015977 target="_blank">GO:0015977</a></span>
<span title="tricarboxylic acid cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006099 target="_blank">GO:0006099</a></span>
<span title="phosphoenolpyruvate carboxylase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0008964 target="_blank">GO:0008964</a></span>
<span title="cytoplasm"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005737 target="_blank">GO:0005737</a></span>
<span title="photosynthesis"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0015979 target="_blank">GO:0015979</a></span>
<span title="lyase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016829 target="_blank">GO:0016829</a></span>
</font></td></tr>
   </table>
<p>
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
     <tr ><th colspan=6 align="left"><font face="Arial" size="3"><br>Consensus prediction of GO terms</font><th><td>&nbsp;</td></tr>
      <tr><td>
      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Molecular Function</b></font></td>
      <td align="center"><font face="Arial"><span title="glutamate-ammonia ligase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004356 target="_blank">GO:0004356</a></span></font></td>
      <td align="center"><font face="Arial"><span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span></font></td>
      <td align="center"><font face="Arial"><span title="GTPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003924 target="_blank">GO:0003924</a></span></font></td>
      <td align="center"><font face="Arial"><span title="protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005515 target="_blank">GO:0005515</a></span></font></td>
      <td align="center"><font face="Arial"><span title="GTP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005525 target="_blank">GO:0005525</a></span></font></td>
      <td align="center"><font face="Arial"><span title="structural constituent of cytoskeleton"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005200 target="_blank">GO:0005200</a></span></font></td>
      <td align="center"><font face="Arial"><span title="alpha-L-fucosidase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004560 target="_blank">GO:0004560</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.10</font></td>
      <td align="center"><font face="Arial">0.10</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Biological Process</b></font></td>
      <td align="center"><font face="Arial"><span title="GTP catabolic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006184 target="_blank">GO:0006184</a></span></font></td>
      <td align="center"><font face="Arial"><span title="meiotic spindle organization"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000212 target="_blank">GO:0000212</a></span></font></td>
      <td align="center"><font face="Arial"><span title="G2/M transition of mitotic cell cycle"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000086 target="_blank">GO:0000086</a></span></font></td>
      <td align="center"><font face="Arial"><span title="protein polymerization"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0051258 target="_blank">GO:0051258</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Cellular Component</b></font></td>
      <td align="center"><font face="Arial"><span title="extracellular region"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005576 target="_blank">GO:0005576</a></span></font></td>
      <td align="center"><font face="Arial"><span title="apical part of cell"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0045177 target="_blank">GO:0045177</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cell leading edge"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0031252 target="_blank">GO:0031252</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cytoplasmic microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005881 target="_blank">GO:0005881</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cytosol"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005829 target="_blank">GO:0005829</a></span></font></td>
      <td align="center"><font face="Arial"><span title="centriole"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005814 target="_blank">GO:0005814</a></span></font></td>
      <td align="center"><font face="Arial"><span title="polar microtubule"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005827 target="_blank">GO:0005827</a></span></font></td>
      <td align="center"><font face="Arial"><span title="gamma-tubulin complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000930 target="_blank">GO:0000930</a></span></font></td>
      <td align="center"><font face="Arial"><span title="pericentriolar material"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000242 target="_blank">GO:0000242</a></span></font></td>
      <td align="center"><font face="Arial"><span title="condensed nuclear chromosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0000794 target="_blank">GO:0000794</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.10</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
      <td align="center"><font face="Arial">0.09</font></td>
  </tr>
   </table>
  </td></tr></table>
<p>
       <table  style="width:1200px;font-size:12px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is a combined measure for evaluating global and local similarity between query and template protein. It's range is [0-1] and higher values indicate more confident predictions.</td></tr>
      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
      <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consensus GO terms amongst the top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template.</td></tr>
     </table>
  </div>
</td></tr>
<tr><td><br/></td></tr>
<tr><td>
<h3 align="center">[Click on <a href="S550642_results.tar.bz2" download>S550642_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<hr><br/>
<div style="position:relative;left:5px;"><table style="font-family:Arial;font-size:14px;"><tr><td colspan=2>Please cite the following articles when you use the I-TASSER server:</td><tr>
<tr><td valign="top">1.</td><td>Yang Zhang. I-TASSER: Fully automated protein structure prediction in CASP8. Proteins, 77 (Suppl 9): 100-113, 2009.</td></tr>
<tr><td valign="top">2.</td><td>Ambrish Roy, Jianyi Yang, Yang Zhang. COFACTOR: an accurate comparative algorithm for structure-based protein function annotation. Nucleic Acids Research, 40: W471-W477, 2012.</td></tr>
<tr><td valign="top">3.</td><td> Jianyi Yang, Yang Zhang. I-TASSER server: new development for protein structure and function predictions, Nucleic Acids Research, 43: W174-W181, 2015.</td></tr>
</table></div><br></td></tr>
</table></body>
</html>
