<html>
<head>
  <title>I-TASSER results</title>
</head>

<script type="text/javascript" src="../../bin/jmol/JSmol.min.js"></script>
<script type="text/javascript" src="../../bin/jmol/Jmol2.js"></script>
<body bgcolor="#FFFFFF">
<font face="Times New Roman">
[<a href="http://zhanglab.ccmb.med.umich.edu">Home</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER">Server</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/queue.php">Queue</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/about.html">About</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/remove.html">Remove</a>]
[<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/statistics.html">Statistics</a>]
<br/>
<table border="0" style="table-layout:fixed;" width="100%" cellpadding="2" cellspaceing="0">
<tr><td>
  <h1 align="center">I-TASSER results for job id S551597</h1>
<center> (Click on <a href="S551597_results.tar.bz2" download>S551597_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page. Click on <a href=http://zhanglab.ccmb.med.umich.edu/I-TASSER/annotation target=_blank>Annotation of I-TASSER Output</a> to read the instructions for how to interpret the results on this page. Model results are kept on the server for 60 days, there is no way to retrieve the modeling data older than 2 months)
</center><br>
</td></tr>
<tr><td>
   <div style="background:#6599FF;width:350px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Submitted Sequence in <a href=http://zhanglab.ccmb.med.umich.edu/FASTA/ target="_blank">FASTA format</a></font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
   <table style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
   <tr><td colspan="2">>protein<br>
MGKHFVLAITDVMPEGVLSDDEMEELKAKLEEALKPLDTVIEVCKFEPVAFGLKKFKMRL<br/>
KIPEELEGGTQPVEDALEAVDGIQRAEIGMVSRI<br/>
</td></tr>
   </table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Secondary Structure</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/>MGKHFVLAITDVMPEGVLSDDEMEELKAKLEEALKPLDTVIEVCKFEPVAFGLKKFKMRLKIPEELEGGTQPVEDALEAVDGIQRAEIGMVSRI</td></tr>
    <tr><td><font="Arial"><span title="Secondary Structure Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">CCCC<font color="blue">SSSSSSSS</font>CCCCCCCCCC<font color="red">HHHHHHHHHHH</font>CCCCCC<font color="blue">SS</font>CCC<font color="blue">SSS</font>C<font color="red">HHHHHHH</font><font color="blue">SSSSSSS</font>CCCCCCCC<font color="red">HHHHHHHH</font>CCCCCC<font color="blue">SSSSSSSS</font>CC</td>
    <tr><td><font="Arial"><span title="Confidence Score of Prediction"><b>Conf.Score</b></span></font></td><td BGCOLOR="#ECECEC">9753899999988899887769999999999652966747532444042430401079999405777774899999964999667999987629</td></tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC"><font color=red>H</font>:Helix; <font color=blue>S</font>:Strand; C:Coil</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted Solvent Accessibility</font>
   </td></tr></table>
   </div>
   <br/>
   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr><td valign="bottom"><font="Arial"><b>Sequence</b></font></td><td BGCOLOR="#DEDEDE">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br/>MGKHFVLAITDVMPEGVLSDDEMEELKAKLEEALKPLDTVIEVCKFEPVAFGLKKFKMRLKIPEELEGGTQPVEDALEAVDGIQRAEIGMVSRI</td></tr>
    <tr><td><font="Arial"><span title="Solvent Accessibility Prediction"><b>Prediction</b></span></font></td><td BGCOLOR="#D7D7D7">7541301010101164453644164026304730446725246265430000131130101023756333440351167174144133343447</td><tr>
    <tr><td><font="Arial"></td><td BGCOLOR="#ECECEC">Values range from <b>0</b> (buried residue) to <b>9</b> (highly exposed residue)</td><tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:300px;">
   <table><tr><td valign="middle">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Predicted normalized B-factor</font>
   </td></tr></table>
   </div>
   <br/>

<font size=2>
(B-factor is a value to indicate the extent of the inherent thermal 

mobility of residues/atoms in proteins. In I-TASSER, this value is 

deduced from threading template proteins from the PDB in combination with the 

sequence profiles derived from sequence databases. The reported B-factor 

profile in the figure below corresponds to the normalized B-factor of the 

target protein, defined by B=(B'-u)/s, where B' is the raw B-factor 

value, u and s are respectively the mean and standard deviation of the raw 

B-factors along the sequence. 

<a href="lscore.txt" target="_blank"  >Click here to read more about predicted 

normalized B-factor</a>) 

</font> 

<br/> 

   <br/> 

   <div style="position:relative;left:30px;">
    <table  border="0" style="font-family:Monospace;font-size:14px;background:#F2F2F2;">
    <tr> <td align =center > <img src=BFP.png>  </td>  </tr>
    </table>
  </div>
  </td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Top 10 threading templates used by I-TASSER</font>
   </td></tr></table>
   </div>
   <br/>
   
<font size=2> 

(I-TASSER modeling starts from the structure templates identified by LOMETS from the PDB library. 

LOMETS is a meta-server threading approach containing multiple threading programs, 

where each threading program can generate tens of thousands of 

template alignments. I-TASSER only uses the templates of the highest significance in the threading alignments, 

the significance of which are measured by the Z-score, i.e. the difference between the raw and average scores 

in the unit of standard deviation. The templates in this section are the 10 best templates selected from 

the LOMETS threading programs. Usually, one template of the highest Z-score is selected from each threading program, where the threading programs are sorted by the average performance in the large-scale benchmark test experiments.) 

</font> 

   <br/> 

   <br/> 

   <div style="position:relative;left:30px;">
   <table style="font-size:13px;background:#F2F2F2;">
      <tr> <th align="left" valign="top"><font face="Arial">Rank</font></th> <th align="center" valign="top"><font face="Arial">PDB<br/>Hit</font></th><th align="center" valign="top"><font face="Arial">Iden1</font></th><th align="center" valign="top"><font face="Arial">Iden2</font></th><th valign="top"><font face="Arial">Cov</font></th><th valign="top"><font face="Arial">Norm.<br/>Z-score</font></th><th valign="top"><font face="Arial">Download<br/>Align.</font></th><td>&nbsp;</td><td BGCOLOR="#DEDEDE" style="font-family:Monospace;font-size:13px;">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;20&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;60&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td></tr>
      <tr><td colspan="7"></td><th align="right" valign ="bottom" ><font face="Arial">Sec.Str<br/>Seq</font></th><td BGCOLOR="#D7D7D7" style="font-family:Monospace;font-size:13px;">CCCC<font color="blue">SSSSSSSS</font>CCCCCCCCCC<font color="red">HHHHHHHHHHH</font>CCCCCC<font color="blue">SS</font>CCC<font color="blue">SSS</font>C<font color="red">HHHHHHH</font><font color="blue">SSSSSSS</font>CCCCCCCC<font color="red">HHHHHHHH</font>CCCCCC<font color="blue">SSSSSSSS</font>CC<br/>MGKHFVLAITDVMPEGVLSDDEMEELKAKLEEALKPLDTVIEVCKFEPVAFGLKKFKMRLKIPEELEGGTQPVEDALEAVDGIQRAEIGMVSRI</td></tr>       <tr><td align="center"><font face="Arial">1</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8A</a></td><td align ="center"><font face="Arial"> 0.35</font></td><td><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 3.07</font></td><td align="center"><a href="threading1.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color="#FFCC00">M</font><font color="#666666">G</font>--D<font color="#0F820F">V</font>V<font color="#333333">A</font>TIK<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font><font color="#E60A0A">E</font>SP--<font color="#E60A0A">D</font>VDL<font color="#E60A0A">E</font>A<font color="#0F820F">L</font><font color="#145AFF">K</font>KEIQ<font color="#E60A0A">E</font>RI-<font color="#D69682">P</font>EG<font color="#FA9600">T</font>ELHKIDE<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VALNVMVVVGDA-<font color="#E60A0A">E</font><font color="#666666">G</font><font color="#666666">G</font><font color="#FA9600">T</font>EAA<font color="#E60A0A">E</font>ES<font color="#0F820F">L</font>SGIE<font color="#666666">G</font>VSNI<font color="#E60A0A">E</font>VTD<font color="#0F820F">V</font>R<font color="#145AFF">R</font>L</td></tr>
       <tr><td align="center"><font face="Arial">2</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3A</a></td><td align ="center"><font face="Arial"> 0.27</font></td><td><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.95</font></td><td align="center"><font face="Arial"> 2.09</font></td><td align="center"><a href="threading2.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-SDFNLVGVIR<font color="#0F820F">V</font>-<font color="#D69682">P</font>TDPDVN--LD<font color="#E60A0A">E</font><font color="#0F820F">L</font>EE<font color="#145AFF">K</font><font color="#0F820F">L</font>KKVI-<font color="#D69682">P</font>EKYGLAKVER<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VAL<font color="#145AFF">K</font>FYVLGRD<font color="#E60A0A">E</font>EGYSFDE<font color="#0F820F">V</font>AEKF<font color="#E60A0A">E</font>E<font color="#0F820F">V</font>ENVES<font color="#333333">A</font><font color="#E60A0A">E</font>VET<font color="#0F820F">V</font><font color="#FA9600">S</font><font color="#145AFF">R</font><font color="#0F820F">I</font></td></tr>
       <tr><td align="center"><font face="Arial">3</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3B</a></td><td align ="center"><font face="Arial"> 0.27</font></td><td><font face="Arial"> 0.29</font></td><td><font face="Arial"> 0.97</font></td><td align="center"><font face="Arial"> 2.48</font></td><td align="center"><a href="threading3.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color="#FFCC00">M</font>SDFNLVGVIR<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font>TDPDVN--LD<font color="#E60A0A">E</font><font color="#0F820F">L</font>EE<font color="#145AFF">K</font><font color="#0F820F">L</font>KKVIPE-KYGLAKVER<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VAL<font color="#145AFF">K</font>FYVLGRD<font color="#E60A0A">E</font>EGYSFDE<font color="#0F820F">V</font>AEKF<font color="#E60A0A">E</font>E<font color="#0F820F">V</font>ENVES<font color="#333333">A</font><font color="#E60A0A">E</font>VET<font color="#0F820F">V</font><font color="#FA9600">S</font><font color="#145AFF">R</font><font color="#0F820F">I</font></td></tr>
       <tr><td align="center"><font face="Arial">4</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8</a></td><td align ="center"><font face="Arial"> 0.31</font></td><td><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 4.73</font></td><td align="center"><a href="threading4.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--MGD<font color="#0F820F">V</font>V<font color="#333333">A</font>TIK<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font><font color="#E60A0A">E</font>SPDV--DL<font color="#E60A0A">E</font>A<font color="#0F820F">L</font><font color="#145AFF">K</font>KEIQ<font color="#E60A0A">E</font>RIPE-G<font color="#FA9600">T</font>ELHKIDE<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VALNVMVVVGDA-<font color="#E60A0A">E</font><font color="#666666">G</font><font color="#666666">G</font><font color="#FA9600">T</font>EAA<font color="#E60A0A">E</font>ES<font color="#0F820F">L</font>SGIE<font color="#666666">G</font>VSNI<font color="#E60A0A">E</font>VTD<font color="#0F820F">V</font>R<font color="#145AFF">R</font>L</td></tr>
       <tr><td align="center"><font face="Arial">5</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8</a></td><td align ="center"><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 3.47</font></td><td align="center"><a href="threading5.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--MGD<font color="#0F820F">V</font>V<font color="#333333">A</font>TIK<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font><font color="#E60A0A">E</font>SPDV<font color="#E60A0A">D</font>--L<font color="#E60A0A">E</font>A<font color="#0F820F">L</font><font color="#145AFF">K</font>KEIQ<font color="#E60A0A">E</font>RI-<font color="#D69682">P</font>EG<font color="#FA9600">T</font>ELHKIDE<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VALNVMVVVGDA-<font color="#E60A0A">E</font><font color="#666666">G</font><font color="#666666">G</font><font color="#FA9600">T</font>EAA<font color="#E60A0A">E</font>ES<font color="#0F820F">L</font>SGIE<font color="#666666">G</font>VSNI<font color="#E60A0A">E</font>VTD<font color="#0F820F">V</font>R<font color="#145AFF">R</font>L</td></tr>
       <tr><td align="center"><font face="Arial">6</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8A</a></td><td align ="center"><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.93</font></td><td align="center"><font face="Arial"> 3.14</font></td><td align="center"><a href="threading6.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">---GD<font color="#0F820F">V</font>V<font color="#333333">A</font>TIK<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font><font color="#E60A0A">E</font>SPDV<font color="#E60A0A">D</font>--L<font color="#E60A0A">E</font>A<font color="#0F820F">L</font><font color="#145AFF">K</font>KEIQ<font color="#E60A0A">E</font>RI-<font color="#D69682">P</font>EG<font color="#FA9600">T</font>ELHKIDE<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VALNVMVVVGDA-<font color="#E60A0A">E</font><font color="#666666">G</font><font color="#666666">G</font><font color="#FA9600">T</font>EAA<font color="#E60A0A">E</font>ES<font color="#0F820F">L</font>SGIE<font color="#666666">G</font>VSNI<font color="#E60A0A">E</font>VTD<font color="#0F820F">V</font>R<font color="#145AFF">R</font>L</td></tr>
       <tr><td align="center"><font face="Arial">7</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3</a></td><td align ="center"><font face="Arial"> 0.27</font></td><td><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 5.82</font></td><td align="center"><a href="threading7.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">-SDFNLVGVIR<font color="#0F820F">V</font>-<font color="#D69682">P</font>TD--P<font color="#E60A0A">D</font>VNLD<font color="#E60A0A">E</font><font color="#0F820F">L</font>EE<font color="#145AFF">K</font><font color="#0F820F">L</font>KK-VI<font color="#D69682">P</font>EKYGLAKVER<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VAL<font color="#145AFF">K</font>FYVLGRD<font color="#E60A0A">E</font>EGYSFDE<font color="#0F820F">V</font>AEKF<font color="#E60A0A">E</font>E<font color="#0F820F">V</font>ENVES<font color="#333333">A</font><font color="#E60A0A">E</font>VET<font color="#0F820F">V</font><font color="#FA9600">S</font><font color="#145AFF">R</font>-</td></tr>
       <tr><td align="center"><font face="Arial">8</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1b64" target="_blank">1b64A</a></td><td align ="center"><font face="Arial"> 0.22</font></td><td><font face="Arial"> 0.29</font></td><td><font face="Arial"> 0.96</font></td><td align="center"><font face="Arial"> 1.35</font></td><td align="center"><a href="threading8.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color="#FFCC00">M</font>LVAKSSILL<font color="#E60A0A">D</font><font color="#0F820F">V</font>K<font color="#D69682">P</font>WDDET<font color="#E60A0A">D</font>--<font color="#FFCC00">M</font>AK<font color="#0F820F">L</font>EECVR-SIQADGL<font color="#0F820F">V</font>WGSS<font color="#145AFF">K</font>LV<font color="#D69682">P</font><font color="#0F820F">V</font>GY<font color="#666666">G</font>I<font color="#145AFF">K</font><font color="#145AFF">K</font>LQIQCVVEDD-KV<font color="#666666">G</font><font color="#FA9600">T</font>DML<font color="#E60A0A">E</font>EQIT<font color="#333333">A</font>FEDV<font color="#00DCDC">Q</font>SMDVAAFNK<font color="#0F820F">I</font></td></tr>
       <tr><td align="center"><font face="Arial">9</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8A</a></td><td align ="center"><font face="Arial"> 0.35</font></td><td><font face="Arial"> 0.33</font></td><td><font face="Arial"> 0.94</font></td><td align="center"><font face="Arial"> 1.00</font></td><td align="center"><a href="threading9.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;"><font color="#FFCC00">M</font><font color="#666666">G</font>D--<font color="#0F820F">V</font>V<font color="#333333">A</font>TIK<font color="#0F820F">V</font><font color="#FFCC00">M</font><font color="#D69682">P</font><font color="#E60A0A">E</font>S--P<font color="#E60A0A">D</font>VDL<font color="#E60A0A">E</font>A<font color="#0F820F">L</font><font color="#145AFF">K</font>KEIQ<font color="#E60A0A">E</font>RI-<font color="#D69682">P</font>EG<font color="#FA9600">T</font>ELHKIDE<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VALNVMVVVGDA-<font color="#E60A0A">E</font><font color="#666666">G</font><font color="#666666">G</font><font color="#FA9600">T</font>EAA<font color="#E60A0A">E</font>ES<font color="#0F820F">L</font>SGIE<font color="#666666">G</font>VSNI<font color="#E60A0A">E</font>VTD<font color="#0F820F">V</font>R<font color="#145AFF">R</font>L</td></tr>
       <tr><td align="center"><font face="Arial">10</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3A</a></td><td align ="center"><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.28</font></td><td><font face="Arial"> 0.95</font></td><td align="center"><font face="Arial"> 2.12</font></td><td align="center"><a href="threading10.pdb" download><font face="Arial">Download</font></a></td><td>&nbsp;</td><td BGCOLOR="#ECECEG" style="font-family:Monospace;font-size:13px;">--SD<font color="#3232AA">F</font>N<font color="#0F820F">L</font>VGVIRV<font color="#D69682">P</font>TDPDVN--LD<font color="#E60A0A">E</font><font color="#0F820F">L</font>EE<font color="#145AFF">K</font><font color="#0F820F">L</font>KKVI-<font color="#D69682">P</font>EKYGLAKVER<font color="#E60A0A">E</font><font color="#D69682">P</font>I<font color="#333333">A</font><font color="#3232AA">F</font><font color="#666666">G</font><font color="#0F820F">L</font>VAL<font color="#145AFF">K</font>FYVLGRD<font color="#E60A0A">E</font>EGYSFDE<font color="#0F820F">V</font>AEKF<font color="#E60A0A">E</font>E<font color="#0F820F">V</font>ENVES<font color="#333333">A</font><font color="#E60A0A">E</font>VET<font color="#0F820F">V</font><font color="#FA9600">S</font><font color="#145AFF">R</font><font color="#0F820F">I</font></td></tr>
      <tr><td colspan="9" align="left">
        <table cellspacing="2" border="0" width="1000px" style="font-family:Arial;font-size:12px;">
        <tr><td valign="top">(a)</td><td align="justify">All the residues are colored in black; however, those residues in template which are identical to the residue in the query sequence are highlighted in color. Coloring scheme is based on the property of amino acids, where polar are brightly coloured while non-polar residues are colored in dark shade. (<a href="http://zhanglab.ccmb.med.umich.edu/I-TASSER/color.html" target="_blank">more about the colors used</a>)</td></tr>
        <tr><td valign="top">(b)</td><td align="justify">Rank of templates represents the top ten threading templates used by I-TASSER.</td></tr>
        <tr><td valign="top">(c)</td><td align="justify">Ident1 is the percentage sequence identity of the templates in the threading aligned region with the query sequence.</td></tr>
        <tr><td valign="top">(d)</td><td align="justify">Ident2 is the percentage sequence identity of the whole template chains with  query sequence.</td></tr>
        <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the threading alignment and is equal to the number of aligned residues divided by the length of query protein.</td></tr>
        <tr><td valign="top">(f)</td><td align="justify">Norm. Z-score is the normalized Z-score of the threading alignments. Alignment with a Normalized Z-score >1 mean a good alignment and vice versa.</td></tr>
        <tr><td valign="top">(g)</td><td align="justify">Download Align. provides the 3D structure of the aligned regions of the threading templates.</td></tr>
        <tr><td valign="top">(h)</td><td align="justify">The top 10 alignments reported above (in order of their ranking) are from the following threading programs:</td></tr>
        <tr><td>&nbsp;</td><td align ="left">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
1: MUSTER&nbsp;&nbsp;&nbsp;2: FFAS-3D&nbsp;&nbsp;&nbsp;3: SPARKS-X&nbsp;&nbsp;&nbsp;4: HHSEARCH2&nbsp;&nbsp;&nbsp;5: HHSEARCH I&nbsp;&nbsp;&nbsp;6: Neff-PPAS&nbsp;&nbsp;&nbsp;7: HHSEARCH&nbsp;&nbsp;&nbsp;8: pGenTHREADER&nbsp;&nbsp;&nbsp;9: PROSPECT2&nbsp;&nbsp;&nbsp;10: PRC&nbsp;&nbsp;&nbsp;        </td></tr>
      </table>
      </td></tr></table>
   </div>
</td></tr>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:380px;">
   <table><tr><td style= "valign:middle;">
   <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp; Top 5 final models predicted by I-TASSER </font>
   </td></tr></table>
   </div>
   <br/>
  
<font size=2>

(For each target, I-TASSER simulations generate a large ensemble of
structural conformations, called decoys. To select the final models, I-TASSER uses the SPICKER program to cluster
all the decoys based on the pair-wise structure similarity, and reports up to five models which
corresponds to the five largest structure clusters. The confidence of each model is quantitatively
measured by C-score that is calculated based on the significance of threading template alignments
and the convergence parameters of the structure assembly simulations.
C-score is typically in the range of [-5, 2], where a C-score of a higher value signifies a model with a higher 
confidence and vice-versa. TM-score and RMSD are estimated based on C-score and protein length following 
the correlation observed between these qualities. Since the top 5 models are ranked by the cluster size,
it is possible that the lower-rank models have a higher C-score in 
rare cases. Although the first model has a better
quality in most cases, it is also possible that the lower-rank models have a better quality than the
higher-rank models as seen in our benchmark tests. If the I-TASSER simulations converge, it
is possible to have less than 5 clusters generated; this is usually an indication that the
models have a good quality because of the converged simulations.)
<ul>
<li><a href=cscore.txt target="_blank">More about C-score</a>
<li><a href=lscore.html>Local structure accuracy profile of the top five models</a>
</ul>
</font>
<ul>   <div style="position:relative;left:1px;">
       <table  style="table-layout:fixed;width:1780px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
  <tr> <td colspan=1> <font size=2>(By right-click on the images, you can export image file or change the configurations, e.g. modifying the background color or stopping the spin of
your models)</font></td></tr>
       <tr>
       <td align ="left"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(350, "load model1.pdb; calculate structure rama; set frank off; select all; color group; spacefill off; wireframe off; cartoons;set ambient 40;set spin x 10; set spin y 10; spin on;");jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin off", "spin off", "Spin On/Off", false)</script></td>
       </tr> <tr> <td  align =left>
<li><a href="model1.pdb" download>Download Model 1</a>

<li>C-score=0.51 (<a href="cscore.txt" target="_blank" >Read more about C-score</a>) 

<li>Estimated TM-score = 0.78&plusmn;0.10

<li>Estimated RMSD = 2.9&plusmn;2.1&Aring;

<br><br>

</td>
       </tr>
       </table>
    </div>
    </div>
  </td></tr>
  </ul>
  <tr><td>
   <br/>
   <div style="background:#6599FF;width:670px;">
      <table>
         <tr><td style= "valign:middle;"><font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Proteins structurally close to the target in the PDB (as identified by <a href="http://zhanglab.ccmb.med.umich.edu/TM-align" target="_blank">TM-align</a>)</font></td></tr>
      </table>
   </div>
   <br/>
   
<font size=2>

(After the structure assembly simulation, I-TASSER uses the TM-align structural alignment program to match the first I-TASSER model to all structures in the PDB library. This section reports the top 10 proteins from the PDB that have the closest structural similarity, 

i.e. the highest <a href=http://zhanglab.ccmb.med.umich.edu/TM-score>TM-score</a>, to the predicted I-TASSER model. Due to the structural similarity, these proteins often have similar function to the target. However, users are encouraged to use the data in the next section 'Predicted function using COACH' to infer the function of the target protein, since COACH has been extensively trained to derive biological functions from multi-source of sequence and structure features which has on average a higher accuracy than the function annotations derived only from the global structure comparison.)</font> 

   <br/>

   <br/>

   <div>
   <div style="position:relative;left:30px;background:#F2F2F2;">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
           <tr>
           <td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1_2yy3A.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation"); jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top"><b>Top 10 Identified stuctural analogs in PDB</b><br/><br/>
                 <table border="0" style="font-family:Arial;font-size:13px;">
                    <tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th >Rank</th><th >PDB Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th>Cov</th><th>Alignment</th></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2yy3A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", true);</script></td><font face="Monospace">
                       <td valign="middle" align="center">1</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3A</a></font></td><td align="center">0.860</td><td valign="middle" align="center"> 1.43</td><td valign="middle" align="center">0.270</td><td valign="middle">0.947</td><td align="center"><a href="model1_2yy3A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1f60B.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">2</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1f60" target="_blank">1f60B</a></font></td><td align="center">0.820</td><td valign="middle" align="center"> 1.74</td><td valign="middle" align="center">0.180</td><td valign="middle">0.947</td><td align="center"><a href="model1_1f60B.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2n51A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">3</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2n51" target="_blank">2n51A</a></font></td><td align="center">0.798</td><td valign="middle" align="center"> 1.83</td><td valign="middle" align="center">0.213</td><td valign="middle">0.936</td><td align="center"><a href="model1_2n51A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1b64A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">4</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1b64" target="_blank">1b64A</a></font></td><td align="center">0.783</td><td valign="middle" align="center"> 1.99</td><td valign="middle" align="center">0.222</td><td valign="middle">0.957</td><td align="center"><a href="model1_1b64A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1lfwA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">5</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1lfw" target="_blank">1lfwA</a></font></td><td align="center">0.716</td><td valign="middle" align="center"> 2.69</td><td valign="middle" align="center">0.065</td><td valign="middle">0.979</td><td align="center"><a href="model1_1lfwA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1i95F.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">6</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1i95" target="_blank">1i95F</a></font></td><td align="center">0.707</td><td valign="middle" align="center"> 2.33</td><td valign="middle" align="center">0.131</td><td valign="middle">0.894</td><td align="center"><a href="model1_1i95F.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2vihA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">7</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2vih" target="_blank">2vihA</a></font></td><td align="center">0.706</td><td valign="middle" align="center"> 2.32</td><td valign="middle" align="center">0.103</td><td valign="middle">0.925</td><td align="center"><a href="model1_2vihA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_1gh8A.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">8</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8A</a></font></td><td align="center">0.703</td><td valign="middle" align="center"> 2.50</td><td valign="middle" align="center">0.250</td><td valign="middle">0.936</td><td align="center"><a href="model1_1gh8A.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr ><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_4fvmA.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">9</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4fvm" target="_blank">4fvmA</a></font></td><td align="center">0.701</td><td valign="middle" align="center"> 2.51</td><td valign="middle" align="center">0.048</td><td valign="middle">0.894</td><td align="center"><a href="model1_4fvmA.pdb" download><font face="Arial">Download</font></a></td></tr>
               
                  <tr bgcolor="#DEDEDE"><td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load model1_2xmaE.pdb; set frank off; select all; spacefill off; wireframe off; select :T; trace 0.35; color trace mediumpurple; select :Q; cartoons; color group; set ambient 40;     set spin x 5; set spin y 5; spin off;", "", false);</script></td><font face="Monospace">
                       <td valign="middle" align="center">10</td><td valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2xma" target="_blank">2xmaE</a></font></td><td align="center">0.700</td><td valign="middle" align="center"> 2.35</td><td valign="middle" align="center">0.069</td><td valign="middle">0.925</td><td align="center"><a href="model1_2xmaE.pdb" download><font face="Arial">Download</font></a></td></tr>
               </table><br/>
               <table border="0" cellspacing="2" style="font-family:Arial;font-size:12px;background:#F2F2F2;">
                <tr><td valign="top">(a)</td><td align="justify">Query structure is shown in cartoon, while the structural analog is displayed using backbone trace.</td></tr>
                <tr><td valign="top">(b)</td><td align="justify">Ranking of proteins is based on TM-score of the structural alignment between the query structure and known structures in the PDB library.</td></tr>
                <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
                <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
                <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of the alignment by TM-align and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
               </table>
          </td></tr>
       </table>
    </div>
    </div>
  </td></tr>
  <tr><td>
  <br/><br/>
   <div style="background:#6599FF;width:450px;">
   <table><tr><td style= "valign:middle;">
      <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Predicted function using <a href="http://zhanglab.ccmb.med.umich.edu/COFACTOR" target="_blank">COFACTOR</a> and <a href="http://zhanglab.ccmb.med.umich.edu/COACH" target="_blank">COACH</a></font>
   </td></tr></table>
   </div>
  <br/>
  
<font size=2>

(This section reports biological annotations of the target 
protein by COFACTOR and COACH based on the I-TASSER structure prediction. 
While COFACTOR deduces protein functions (ligand-binding sites, EC and GO)
using structure comparison and protein-protein networks,
COACH is a meta-server approach that combines multiple function
annotation results (on ligand-binding sites) from the COFACTOR, 
TM-SITE and S-SITE programs.)

</font>

<br/>

<br/>

  <tr><td>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Ligand binding sites</font>
      </td></tr></table>
   </div>
  <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td valign="top" align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load CH_TM_4dh9F_BS01_NUC.pdb; calculate structure rama; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 6,8,70,71,75,78,85,87,88,90,91; wireframe 75; spacefill 100; color blue;select 6 and *.ca; label %m%R; color label magenta;select 8 and *.ca; label %m%R; color label magenta;select 70 and *.ca; label %m%R; color label magenta;select 71 and *.ca; label %m%R; color label magenta;select 75 and *.ca; label %m%R; color label magenta;select 78 and *.ca; label %m%R; color label magenta;select 85 and *.ca; label %m%R; color label magenta;select 87 and *.ca; label %m%R; color label magenta;select 88 and *.ca; label %m%R; color label magenta;select 90 and *.ca; label %m%R; color label magenta;select 91 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow;  cartoons; wireframe off; spacefill off; set ambient 40;     set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT> jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("set spin x 10; set spin y 10; spin on", "spin off", "Spin On/Off", false)</SCRIPT>
</td>
       <td valign="top">       <table border="0"  style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"><th>Click<br>to view</th><th>Rank</th><th>C-score</th><th>Cluster<br>size</th> <th>PDB<br>Hit</th> <th>Lig<br>Name</th> <th>Download<br>Complex</th> <th align=left>Ligand Binding Site Residues</th> </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_4dh9F_BS01_NUC.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 6,8,70,71,75,78,85,87,88,90,91; color blue; wireframe 75; spacefill 100;select 6 and *.ca; label %m%R; color label magenta;select 8 and *.ca; label %m%R; color label magenta;select 70 and *.ca; label %m%R; color label magenta;select 71 and *.ca; label %m%R; color label magenta;select 75 and *.ca; label %m%R; color label magenta;select 78 and *.ca; label %m%R; color label magenta;select 85 and *.ca; label %m%R; color label magenta;select 87 and *.ca; label %m%R; color label magenta;select 88 and *.ca; label %m%R; color label magenta;select 90 and *.ca; label %m%R; color label magenta;select 91 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow;  cartoons; wireframe off; spacefill off; set ambient 40;    set spin x 5; set spin y 5; spin off;", "", true);</script></td>
    <font face="Monospace"><td valign="middle" align="center">1</td><td valign="middle" align="center">0.15</td> <td valign="middle" align="center">12</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=4dh9 target="_blank">4dh9F</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=NUC target="_blank">Nuc.Acid</a> </td>  <td valign="middle" align=center><a href="CH_TM_4dh9F_BS01_NUC.pdb" download>Rep</a>, <a href="CH_complex1.pdb" download>Mult</a></td>  <td valign="middle" align="left">6,8,70,71,75,78,85,87,88,90,91</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_2qc8G_BS01_MN.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 9,11,47,57,90; color blue; wireframe 75; spacefill 100;select 9 and *.ca; label %m%R; color label magenta;select 11 and *.ca; label %m%R; color label magenta;select 47 and *.ca; label %m%R; color label magenta;select 57 and *.ca; label %m%R; color label magenta;select 90 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">2</td><td valign="middle" align="center">0.14</td> <td valign="middle" align="center">11</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=2qc8 target="_blank">2qc8G</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=MN target="_blank">MN</a> </td>  <td valign="middle" align=center><a href="CH_TM_2qc8G_BS01_MN.pdb" download>Rep</a>, <a href="CH_complex2.pdb" download>Mult</a></td>  <td valign="middle" align="left">9,11,47,57,90</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_TM_1y10A_BS01_CA.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 11,12,13,56,87; color blue; wireframe 75; spacefill 100;select 11 and *.ca; label %m%R; color label magenta;select 12 and *.ca; label %m%R; color label magenta;select 13 and *.ca; label %m%R; color label magenta;select 56 and *.ca; label %m%R; color label magenta;select 87 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">3</td><td valign="middle" align="center">0.12</td> <td valign="middle" align="center">9</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=1y10 target="_blank">1y10A</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=CA target="_blank">CA</a> </td>  <td valign="middle" align=center><a href="CH_TM_1y10A_BS01_CA.pdb" download>Rep</a>, <a href="CH_complex3.pdb" download>Mult</a></td>  <td valign="middle" align="left">11,12,13,56,87</td>    </tr>
       <tr bgcolor="#DEDEDE">
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_1wc1A_BS01_TAT.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 13,14,16,17,18,47,48,55; color blue; wireframe 75; spacefill 100;select 13 and *.ca; label %m%R; color label magenta;select 14 and *.ca; label %m%R; color label magenta;select 16 and *.ca; label %m%R; color label magenta;select 17 and *.ca; label %m%R; color label magenta;select 18 and *.ca; label %m%R; color label magenta;select 47 and *.ca; label %m%R; color label magenta;select 48 and *.ca; label %m%R; color label magenta;select 55 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">4</td><td valign="middle" align="center">0.03</td> <td valign="middle" align="center">3</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=1wc1 target="_blank">1wc1A</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=TAT target="_blank">TAT</a> </td>  <td valign="middle" align=center><a href="CH_CF_1wc1A_BS01_TAT.pdb" download>Rep</a>, <a href="CH_complex4.pdb" download>Mult</a></td>  <td valign="middle" align="left">13,14,16,17,18,47,48,55</td>    </tr>
       <tr >
       <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("load CH_CF_4ursA_BS02_C2E.pdb; set frank off; select all; cpk off; spacefill off; wireframe off; cartoons;  set ambient 40; select 35,60,61,62,63,69,73,76; color blue; wireframe 75; spacefill 100;select 35 and *.ca; label %m%R; color label magenta;select 60 and *.ca; label %m%R; color label magenta;select 61 and *.ca; label %m%R; color label magenta;select 62 and *.ca; label %m%R; color label magenta;select 63 and *.ca; label %m%R; color label magenta;select 69 and *.ca; label %m%R; color label magenta;select 73 and *.ca; label %m%R; color label magenta;select 76 and *.ca; label %m%R; color label magenta;select hetero; color greenyellow; set radius 0.90; set frank off; spacefill; wireframe off;    set spin x 5; set spin y 5; spin off;", "", false);</script></td>
    <font face="Monospace"><td valign="middle" align="center">5</td><td valign="middle" align="center">0.02</td> <td valign="middle" align="center">2</td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/qsearch_pdb.cgi?pdbid=4urs target="_blank">4ursA</a></td> <td valign="middle" align="left"><a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/sym.cgi?code=C2E target="_blank">C2E</a> </td>  <td valign="middle" align=center><a href="CH_CF_4ursA_BS02_C2E.pdb" download>Rep</a>, <a href="CH_complex5.pdb" download>Mult</a></td>  <td valign="middle" align="left">35,60,61,62,63,69,73,76</td>    </tr>
       </table>
       <br/>
       <table  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
       <tr><td valign="top"></td><td align="justify"><a href=prob.txt target="_blank">Download</a> the residue-specific ligand binding probability, which is estimated by SVM. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.inf target="_blank">Download</a> the all possible binding ligands and detailed prediction summary. </td></tr> 
       <tr><td valign="top"></td><td align="justify"><a href=Bsites.clr target="_blank">Download</a> the templates clustering results. </td></tr> 
        <tr><td valign="top">(a)</td><td align="justify"><b>C-score</b> is the confidence score of the prediction. C-score ranges [0-1], where a higher score indicates a more reliable prediction. </td></tr>
        <tr><td valign="top">(b)</td><td align="justify"><b>Cluster size</b> is the total number of templates in a cluster. </td></tr>
       <tr><td valign="top">(c)</td><td align="justify"><b>Lig Name</b> is name of possible binding ligand. Click the name to view its information in <a href=http://zhanglab.ccmb.med.umich.edu/BioLiP/>the BioLiP database</a>.</td></tr>
       <tr><td valign="top">(d)</td><td align="justify"><b>Rep</b> is a single complex structure with the most representative ligand in the cluster, i.e., the one listed in the <b>Lig Name</b> column. <br>
 <b>Mult</b> is the complex structures with all potential binding ligands in the cluster.</td></tr>
<br>       </table></td></tr>
     </table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:430px;">
   <table><tr><td style= "valign:middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Enzyme Commission (EC) numbers and active sites</font>
   </td></tr></table>
   </div>
   <br/>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
       <tr><td align ="center"><script type="text/javascript">jmolInitialize("../../bin/jmol/");jmolSetAppletColor("#000000");jmolApplet(400, "load model1.pdb; calculate structure rama; set frank off; select all; spacefill off; wireframe off; cartoons; select 55; wireframe 70; spacefill 100; color cpk;  select 55 and *.ca; label %m%R; color label magenta; select 79; wireframe 70; spacefill 100; color cpk;  select 79 and *.ca; label %m%R; color label magenta;      set spin x 10; set spin y 10; spin off;");</script><br/><SCRIPT>jmolButton("Reset","Reset to initial orientation");  jmolCheckbox("spin on", "spin off", "Spin On/Off", false)</SCRIPT></td>
           <td valign="top">
             <table  border="0" style="font-family:Arial;font-size:13px;"><tr bgcolor="#DEDEDE"> <th>Click<br>to view</th><th >Rank</th><th >Cscore<sup>EC</sup></th><th>PDB<br/>Hit</th><th >TM-score</th><th >RMSD<sup>a</sup></th><th >IDEN<sup>a</sup></th><th >Cov</th><th >EC Number</th><th align=left>Active Site Residues</th></tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 55; wireframe 70; spacefill 100; color cpk;  select 55 and *.ca; label %m%R; color label magenta; select 79; wireframe 70; spacefill 100; color cpk;  select 79 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">1</font></td><td align="center"><font face="Arial">0.373</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3cjk" target="_blank">3cjkA</a></td><td align="center"><font face="Arial">0.604</font></td><td align="center"><font face="Arial"> 1.67</font></td><td align="center"><font face="Arial">0.061</font></td><td align="right"><font face="Arial">0.702</font></td>
         <td align="left"><span title="Copper-exporting ATPase."><a href="http://ca.expasy.org/enzyme/3.6.3.4" target="_blank"><font face="Arial">3.6.3.4</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">55,79</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 30; wireframe 70; spacefill 100; color cpk;  select 30 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">2</font></td><td align="center"><font face="Arial">0.345</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1fvq" target="_blank">1fvqA</a></td><td align="center"><font face="Arial">0.635</font></td><td align="center"><font face="Arial"> 1.84</font></td><td align="center"><font face="Arial">0.127</font></td><td align="right"><font face="Arial">0.755</font></td>
         <td align="left"><span title="Copper-exporting ATPase."><a href="http://ca.expasy.org/enzyme/3.6.3.4" target="_blank"><font face="Arial">3.6.3.4</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">30</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off; select 54; wireframe 70; spacefill 100; color cpk;  select 54 and *.ca; label %m%R; color label magenta;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">3</font></td><td align="center"><font face="Arial">0.344</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1mwy" target="_blank">1mwyA</a></td><td align="center"><font face="Arial">0.623</font></td><td align="center"><font face="Arial"> 2.30</font></td><td align="center"><font face="Arial">0.153</font></td><td align="right"><font face="Arial">0.766</font></td>
         <td align="left"><span title="Cadmium-exporting ATPase."><a href="http://ca.expasy.org/enzyme/3.6.3.3" target="_blank"><font face="Arial">3.6.3.3</font></a></span>&nbsp;<span title="Zinc-exporting ATPase."><a href="http://ca.expasy.org/enzyme/3.6.3.5" target="_blank"><font face="Arial">3.6.3.5</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">54</font></td>         </tr>
         <tr bgcolor="#DEDEDE">
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">4</font></td><td align="center"><font face="Arial">0.316</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1lgr" target="_blank">1lgrA</a></td><td align="center"><font face="Arial">0.685</font></td><td align="center"><font face="Arial"> 2.83</font></td><td align="center"><font face="Arial">0.065</font></td><td align="right"><font face="Arial">0.957</font></td>
         <td align="left"><span title="Glutamate--ammonia ligase."><a href="http://ca.expasy.org/enzyme/6.3.1.2" target="_blank"><font face="Arial">6.3.1.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         <tr >
         <td align="center"><script language="JavaScript" type="text/javascript">jmolRadio("select all; wireframe off; spacefill off; label off;   set spin x 5; set spin y 5; spin off;", "", false);</script></td>
         <td align="left"><font face="Arial">5</font></td><td align="center"><font face="Arial">0.316</font></td><td><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2lgs" target="_blank">2lgsA</a></td><td align="center"><font face="Arial">0.685</font></td><td align="center"><font face="Arial"> 2.84</font></td><td align="center"><font face="Arial">0.065</font></td><td align="right"><font face="Arial">0.957</font></td>
         <td align="left"><span title="Glutamate--ammonia ligase."><a href="http://ca.expasy.org/enzyme/6.3.1.2" target="_blank"><font face="Arial">6.3.1.2</font></a></span>&nbsp;         </td>
         <td align="left"><font face="Arial">NA</font></td>         </tr>
         </table><br/>
         <table border="0"  cellspacing="2" border="0" style="font-family:Arial;font-size:12px;">
          <tr><td valign="top">&nbsp;</td><td align="justify"><b>Click on the radio buttons to visualize predicted active site residues.</b></td></tr>
           <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>EC</sup> is the confidence score for the EC number prediction. Cscore<sup>EC</sup> values range in between [0-1]; <br>where a higher score indicates a more reliable EC number prediction.</td></tr>
           <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
           <tr><td valign="bottom">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
           <tr><td valign="bottom">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
           <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural  alignment and is equal to the number of structurally aligned residues divided <br>by length of the query protein.</td></tr>
         </table>
        </td></tr></table>
  </div>
</td></tr>
  <tr><td>
  <br/>
   <div style="position:relative;left:30px;background:#FF9900; width:280px;">
      <table><tr><td  valign="middle;">
        <font color="#FFFFFF" size="4" face="Arial">&nbsp;&nbsp;Gene Ontology (GO) terms</font>
      </td></tr></table>
   </div>
  <div style="position:relative;left:30px;background:#F2F2F2;width:100%">
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr background:#FF9900;><th colspan=8 align="left"><font face="Arial" size="3">Top 10 homologous GO templates in PDB</font><th><td>&nbsp;</td></tr>
  <tr bgcolor="#DEDEDE"><th ><font face="Arial">Rank</font></th><th><font face="Arial">Cscore<sup>GO</sup></font></th><th><font face="Arial">TM-score</font></th><th><font face="Arial">RMSD<sup>a</sup></font></th><th><font face="Arial">IDEN<sup>a</sup></font></th><th><font face="Arial">Cov</font></th><th align="left"><font face="Arial">PDB Hit</font></th><th align="left"><font face="Arial">Associated GO Terms</font></th></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">1</font></td><td align="center"  valign="middle"><font face="Arial"> 0.46</font></td><td  valign="middle" align="center"><font face="Arial">0.7834</font></td><td align="center"  valign="middle"><font face="Arial"> 1.99</font></td><td align="center"  valign="middle"><font face="Arial"> 0.22</font></td><td  valign="middle" align="center"><font face="Arial"> 0.96</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1b64" target="_blank">1b64A</a></td><td valign="middle"><font face="Arial">
<span title="translation elongation factor activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003746 target="_blank">GO:0003746</a></span>
<span title="eukaryotic translation elongation factor 1 complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005853 target="_blank">GO:0005853</a></span>
<span title="translational elongation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006414 target="_blank">GO:0006414</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">2</font></td><td align="center"  valign="middle"><font face="Arial"> 0.45</font></td><td  valign="middle" align="center"><font face="Arial">0.8202</font></td><td align="center"  valign="middle"><font face="Arial"> 1.74</font></td><td align="center"  valign="middle"><font face="Arial"> 0.18</font></td><td  valign="middle" align="center"><font face="Arial"> 0.95</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1f60" target="_blank">1f60B</a></td><td valign="middle"><font face="Arial">
<span title="translation elongation factor activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003746 target="_blank">GO:0003746</a></span>
<span title="eukaryotic translation elongation factor 1 complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005853 target="_blank">GO:0005853</a></span>
<span title="translational elongation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006414 target="_blank">GO:0006414</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">3</font></td><td align="center"  valign="middle"><font face="Arial"> 0.42</font></td><td  valign="middle" align="center"><font face="Arial">0.8602</font></td><td align="center"  valign="middle"><font face="Arial"> 1.43</font></td><td align="center"  valign="middle"><font face="Arial"> 0.27</font></td><td  valign="middle" align="center"><font face="Arial"> 0.95</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2yy3" target="_blank">2yy3A</a></td><td valign="middle"><font face="Arial">
<span title="translation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006412 target="_blank">GO:0006412</a></span>
<span title="translational elongation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006414 target="_blank">GO:0006414</a></span>
<span title="eukaryotic translation elongation factor 1 complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005853 target="_blank">GO:0005853</a></span>
<span title="translation elongation factor activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003746 target="_blank">GO:0003746</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">4</font></td><td align="center"  valign="middle"><font face="Arial"> 0.40</font></td><td  valign="middle" align="center"><font face="Arial">0.6075</font></td><td align="center"  valign="middle"><font face="Arial"> 1.58</font></td><td align="center"  valign="middle"><font face="Arial"> 0.06</font></td><td  valign="middle" align="center"><font face="Arial"> 0.70</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1fe0" target="_blank">1fe0B</a></td><td valign="middle"><font face="Arial">
<span title="copper ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005507 target="_blank">GO:0005507</a></span>
<span title="ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006811 target="_blank">GO:0006811</a></span>
<span title="metal ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030001 target="_blank">GO:0030001</a></span>
<span title="metallochaperone activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016530 target="_blank">GO:0016530</a></span>
<span title="copper ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006825 target="_blank">GO:0006825</a></span>
<span title="copper chaperone activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016531 target="_blank">GO:0016531</a></span>
<span title="cytosol"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005829 target="_blank">GO:0005829</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="cellular copper ion homeostasis"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006878 target="_blank">GO:0006878</a></span>
<span title="transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006810 target="_blank">GO:0006810</a></span>
<span title="copper-dependent protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032767 target="_blank">GO:0032767</a></span>
<span title="response to oxidative stress"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006979 target="_blank">GO:0006979</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">5</font></td><td align="center"  valign="middle"><font face="Arial"> 0.37</font></td><td  valign="middle" align="center"><font face="Arial">0.6475</font></td><td align="center"  valign="middle"><font face="Arial"> 1.84</font></td><td align="center"  valign="middle"><font face="Arial"> 0.11</font></td><td  valign="middle" align="center"><font face="Arial"> 0.77</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3dxs" target="_blank">3dxsX</a></td><td valign="middle"><font face="Arial">
<span title="copper-exporting ATPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004008 target="_blank">GO:0004008</a></span>
<span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span>
<span title="ATP biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006754 target="_blank">GO:0006754</a></span>
<span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span>
<span title="metal ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030001 target="_blank">GO:0030001</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
<span title="copper ion export"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0060003 target="_blank">GO:0060003</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">6</font></td><td align="center"  valign="middle"><font face="Arial"> 0.35</font></td><td  valign="middle" align="center"><font face="Arial">0.7025</font></td><td align="center"  valign="middle"><font face="Arial"> 2.50</font></td><td align="center"  valign="middle"><font face="Arial"> 0.25</font></td><td  valign="middle" align="center"><font face="Arial"> 0.94</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1gh8" target="_blank">1gh8A</a></td><td valign="middle"><font face="Arial">
<span title="translation elongation factor activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003746 target="_blank">GO:0003746</a></span>
<span title="translational elongation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006414 target="_blank">GO:0006414</a></span>
<span title="eukaryotic translation elongation factor 1 complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005853 target="_blank">GO:0005853</a></span>
<span title="translation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006412 target="_blank">GO:0006412</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">7</font></td><td align="center"  valign="middle"><font face="Arial"> 0.34</font></td><td  valign="middle" align="center"><font face="Arial">0.6349</font></td><td align="center"  valign="middle"><font face="Arial"> 1.84</font></td><td align="center"  valign="middle"><font face="Arial"> 0.13</font></td><td  valign="middle" align="center"><font face="Arial"> 0.76</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1fvq" target="_blank">1fvqA</a></td><td valign="middle"><font face="Arial">
<span title="metal ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030001 target="_blank">GO:0030001</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">8</font></td><td align="center"  valign="middle"><font face="Arial"> 0.34</font></td><td  valign="middle" align="center"><font face="Arial">0.6225</font></td><td align="center"  valign="middle"><font face="Arial"> 2.30</font></td><td align="center"  valign="middle"><font face="Arial"> 0.15</font></td><td  valign="middle" align="center"><font face="Arial"> 0.77</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1mwy" target="_blank">1mwyA</a></td><td valign="middle"><font face="Arial">
<span title="metal ion transport"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030001 target="_blank">GO:0030001</a></span>
<span title="metal ion binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0046872 target="_blank">GO:0046872</a></span>
</font></td></tr>
			<tr ><td align="center" valign="middle"><font face="Arial">9</font></td><td align="center"  valign="middle"><font face="Arial"> 0.34</font></td><td  valign="middle" align="center"><font face="Arial">0.7002</font></td><td align="center"  valign="middle"><font face="Arial"> 2.42</font></td><td align="center"  valign="middle"><font face="Arial"> 0.12</font></td><td  valign="middle" align="center"><font face="Arial"> 0.93</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=2j5a" target="_blank">2j5aA</a></td><td valign="middle"><font face="Arial">
<span title="ribonucleoprotein complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030529 target="_blank">GO:0030529</a></span>
<span title="RNA binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003723 target="_blank">GO:0003723</a></span>
<span title="rRNA binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019843 target="_blank">GO:0019843</a></span>
<span title="ribosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005840 target="_blank">GO:0005840</a></span>
<span title="structural constituent of ribosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003735 target="_blank">GO:0003735</a></span>
<span title="translation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006412 target="_blank">GO:0006412</a></span>
</font></td></tr>
			<tr bgcolor="#DEDEDE"><td align="center" valign="middle"><font face="Arial">10</font></td><td align="center"  valign="middle"><font face="Arial"> 0.33</font></td><td  valign="middle" align="center"><font face="Arial">0.6903</font></td><td align="center"  valign="middle"><font face="Arial"> 2.46</font></td><td align="center"  valign="middle"><font face="Arial"> 0.13</font></td><td  valign="middle" align="center"><font face="Arial"> 0.89</font></td><td align="left"  valign="middle"><a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1fjg" target="_blank">1fjgF</a></td><td valign="middle"><font face="Arial">
<span title="translation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006412 target="_blank">GO:0006412</a></span>
<span title="rRNA binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0019843 target="_blank">GO:0019843</a></span>
<span title="ribosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005840 target="_blank">GO:0005840</a></span>
<span title="ribonucleoprotein complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0030529 target="_blank">GO:0030529</a></span>
<span title="structural constituent of ribosome"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003735 target="_blank">GO:0003735</a></span>
<span title="RNA binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003723 target="_blank">GO:0003723</a></span>
</font></td></tr>
   </table>
<p>
       <table  style="width:1200px;font-size:13px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
     <tr ><th colspan=6 align="left"><font face="Arial" size="3"><br>Consensus prediction of GO terms</font><th><td>&nbsp;</td></tr>
      <tr><td>
      <table cellpadding="2" cellspacing="3" style="font-family:Monospace;font-size:13px;background:#F2F2F2">
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Molecular Function</b></font></td>
      <td align="center"><font face="Arial"><span title="translation elongation factor activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0003746 target="_blank">GO:0003746</a></span></font></td>
      <td align="center"><font face="Arial"><span title="copper chaperone activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016531 target="_blank">GO:0016531</a></span></font></td>
      <td align="center"><font face="Arial"><span title="copper-dependent protein binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0032767 target="_blank">GO:0032767</a></span></font></td>
      <td align="center"><font face="Arial"><span title="ATP binding"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005524 target="_blank">GO:0005524</a></span></font></td>
      <td align="center"><font face="Arial"><span title="copper-exporting ATPase activity"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0004008 target="_blank">GO:0004008</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.83</font></td>
      <td align="center"><font face="Arial">0.40</font></td>
      <td align="center"><font face="Arial">0.40</font></td>
      <td align="center"><font face="Arial">0.37</font></td>
      <td align="center"><font face="Arial">0.37</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Biological Process</b></font></td>
      <td align="center"><font face="Arial"><span title="translational elongation"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006414 target="_blank">GO:0006414</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cellular copper ion homeostasis"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006878 target="_blank">GO:0006878</a></span></font></td>
      <td align="center"><font face="Arial"><span title="response to oxidative stress"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006979 target="_blank">GO:0006979</a></span></font></td>
      <td align="center"><font face="Arial"><span title="ATP biosynthetic process"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0006754 target="_blank">GO:0006754</a></span></font></td>
      <td align="center"><font face="Arial"><span title="copper ion export"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0060003 target="_blank">GO:0060003</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.83</font></td>
      <td align="center"><font face="Arial">0.40</font></td>
      <td align="center"><font face="Arial">0.40</font></td>
      <td align="center"><font face="Arial">0.37</font></td>
      <td align="center"><font face="Arial">0.37</font></td>
  </tr>
       <tr bgcolor="#DEDEDE"><td><font face="Arial"><b>Cellular Component</b></font></td>
      <td align="center"><font face="Arial"><span title="eukaryotic translation elongation factor 1 complex"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005853 target="_blank">GO:0005853</a></span></font></td>
      <td align="center"><font face="Arial"><span title="cytosol"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0005829 target="_blank">GO:0005829</a></span></font></td>
      <td align="center"><font face="Arial"><span title="membrane"><a href=https://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0016020 target="_blank">GO:0016020</a></span></font></td>
</tr>
<tr><td ><font face="Arial"><b>GO-Score</b></font></td>      <td align="center"><font face="Arial">0.83</font></td>
      <td align="center"><font face="Arial">0.40</font></td>
      <td align="center"><font face="Arial">0.37</font></td>
  </tr>
   </table>
  </td></tr></table>
<p>
       <table  style="width:1200px;font-size:12px;background:#F2F2F2;" border="0" cellspacing="5" cellpadding="0" align="left">
      <tr><td valign="top">(a)</td><td align="justify">Cscore<sup>GO</sup> is a combined measure for evaluating global and local similarity between query and template protein. It's range is [0-1] and higher values indicate more confident predictions.</td></tr>
      <tr><td valign="top">(b)</td><td align="justify">TM-score is a measure of  global structural similarity between query and template protein.</td></tr>
      <tr><td valign="top">(c)</td><td align="justify">RMSD<sup>a</sup> is the RMSD between residues that are structurally aligned by TM-align.</td></tr>
      <tr><td valign="top">(d)</td><td align="justify">IDEN<sup>a</sup> is the percentage sequence identity in the structurally aligned region.</td></tr>
      <tr><td valign="top">(e)</td><td align="justify">Cov represents the coverage of global structural alignment and is equal to the number of structurally aligned residues divided by length of the query protein.</td></tr>
      <tr><td valign="top">(f)</td><td align="justify">The second table shows a consensus GO terms amongst the top scoring templates. The GO-Score associated with each prediction is defined as the average weight of the GO term, where the weights are assigned based on Cscore<sup>GO</sup> of the template.</td></tr>
     </table>
  </div>
</td></tr>
<tr><td><br/></td></tr>
<tr><td>
<h3 align="center">[Click on <a href="S551597_results.tar.bz2" download>S551597_results.tar.bz2</a> to download the tarball file including all modeling results listed on this page] </h3>
<hr><br/>
<div style="position:relative;left:5px;"><table style="font-family:Arial;font-size:14px;"><tr><td colspan=2>Please cite the following articles when you use the I-TASSER server:</td><tr>
<tr><td valign="top">1.</td><td>Yang Zhang. I-TASSER: Fully automated protein structure prediction in CASP8. Proteins, 77 (Suppl 9): 100-113, 2009.</td></tr>
<tr><td valign="top">2.</td><td>Ambrish Roy, Jianyi Yang, Yang Zhang. COFACTOR: an accurate comparative algorithm for structure-based protein function annotation. Nucleic Acids Research, 40: W471-W477, 2012.</td></tr>
<tr><td valign="top">3.</td><td> Jianyi Yang, Yang Zhang. I-TASSER server: new development for protein structure and function predictions, Nucleic Acids Research, 43: W174-W181, 2015.</td></tr>
</table></div><br></td></tr>
</table></body>
</html>
